diff -Naur visualization-1.6.7/interactive_markers/src/interactive_markers/interactive_marker_server.py visualization/interactive_markers/src/interactive_markers/interactive_marker_server.py
--- visualization-1.6.7/interactive_markers/src/interactive_markers/interactive_marker_server.py	2011-11-18 02:14:49.000000000 +0100
+++ visualization/interactive_markers/src/interactive_markers/interactive_marker_server.py	2012-04-19 00:10:20.544806414 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 # Copyright (c) 2011, Willow Garage, Inc.
 # All rights reserved.
diff -Naur visualization-1.6.7/interactive_markers/src/interactive_markers/menu_handler.py visualization/interactive_markers/src/interactive_markers/menu_handler.py
--- visualization-1.6.7/interactive_markers/src/interactive_markers/menu_handler.py	2011-11-18 02:14:49.000000000 +0100
+++ visualization/interactive_markers/src/interactive_markers/menu_handler.py	2012-04-19 00:10:20.551473081 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 # Copyright (c) 2011, Willow Garage, Inc.
 # All rights reserved.
diff -Naur visualization-1.6.7/wxpropgrid/find_lib visualization/wxpropgrid/find_lib
--- visualization-1.6.7/wxpropgrid/find_lib	2011-11-18 02:14:49.000000000 +0100
+++ visualization/wxpropgrid/find_lib	2012-04-19 00:10:20.541473080 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import roslib; roslib.load_manifest('wxpropgrid')
 
