diff -Naur executive_smach_visualization-1.0.2/smach_viewer/smach_viewer.py executive_smach_visualization/smach_viewer/smach_viewer.py
--- executive_smach_visualization-1.0.2/smach_viewer/smach_viewer.py	2011-10-20 02:41:04.000000000 +0200
+++ executive_smach_visualization/smach_viewer/smach_viewer.py	2012-04-19 00:10:32.428140195 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 # Copyright (c) 2010, Willow Garage, Inc.
 # All rights reserved.
diff -Naur executive_smach_visualization-1.0.2/smach_viewer/test/introspection_test.py executive_smach_visualization/smach_viewer/test/introspection_test.py
--- executive_smach_visualization-1.0.2/smach_viewer/test/introspection_test.py	2011-10-20 02:41:04.000000000 +0200
+++ executive_smach_visualization/smach_viewer/test/introspection_test.py	2012-04-19 00:10:32.431473528 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import roslib; roslib.load_manifest('smach_viewer')
 import rospy
