diff -Naur ros-1.6.8-orig/bin/rosbag ros-1.6.8/bin/rosbag
--- ros-1.6.8-orig/bin/rosbag	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rosbag	2012-04-19 23:50:44.314158146 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rosboost-cfg ros-1.6.8/bin/rosboost-cfg
--- ros-1.6.8-orig/bin/rosboost-cfg	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rosboost-cfg	2012-04-19 23:50:44.314158146 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2010, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rosclean ros-1.6.8/bin/rosclean
--- ros-1.6.8-orig/bin/rosclean	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rosclean	2012-04-19 23:50:44.314158146 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2010, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/roscore ros-1.6.8/bin/roscore
--- ros-1.6.8-orig/bin/roscore	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/roscore	2012-04-19 23:50:44.314158146 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/roscreate-pkg ros-1.6.8/bin/roscreate-pkg
--- ros-1.6.8-orig/bin/roscreate-pkg	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/roscreate-pkg	2012-04-19 23:50:44.314158146 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/roscreate-stack ros-1.6.8/bin/roscreate-stack
--- ros-1.6.8-orig/bin/roscreate-stack	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/roscreate-stack	2012-04-19 23:50:44.314158146 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rosdep ros-1.6.8/bin/rosdep
--- ros-1.6.8-orig/bin/rosdep	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rosdep	2012-04-19 23:50:44.314158146 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rosgraph ros-1.6.8/bin/rosgraph
--- ros-1.6.8-orig/bin/rosgraph	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rosgraph	2012-04-19 23:50:44.314158146 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/roslaunch ros-1.6.8/bin/roslaunch
--- ros-1.6.8-orig/bin/roslaunch	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/roslaunch	2012-04-19 23:50:44.314158146 +0200
@@ -1,3 +1,3 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 from ros import roslaunch
 roslaunch.main()
diff -Naur ros-1.6.8-orig/bin/roslaunch-deps ros-1.6.8/bin/roslaunch-deps
--- ros-1.6.8-orig/bin/roslaunch-deps	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/roslaunch-deps	2012-04-19 23:50:44.314158146 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/roslaunch-logs ros-1.6.8/bin/roslaunch-logs
--- ros-1.6.8-orig/bin/roslaunch-logs	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/roslaunch-logs	2012-04-19 23:50:44.314158146 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rosmake ros-1.6.8/bin/rosmake
--- ros-1.6.8-orig/bin/rosmake	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rosmake	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
diff -Naur ros-1.6.8-orig/bin/rosmaster ros-1.6.8/bin/rosmaster
--- ros-1.6.8-orig/bin/rosmaster	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rosmaster	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rosmsg ros-1.6.8/bin/rosmsg
--- ros-1.6.8-orig/bin/rosmsg	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rosmsg	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rosnode ros-1.6.8/bin/rosnode
--- ros-1.6.8-orig/bin/rosnode	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rosnode	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rosparam ros-1.6.8/bin/rosparam
--- ros-1.6.8-orig/bin/rosparam	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rosparam	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rosservice ros-1.6.8/bin/rosservice
--- ros-1.6.8-orig/bin/rosservice	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rosservice	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rossrv ros-1.6.8/bin/rossrv
--- ros-1.6.8-orig/bin/rossrv	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rossrv	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rostest ros-1.6.8/bin/rostest
--- ros-1.6.8-orig/bin/rostest	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rostest	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rostopic ros-1.6.8/bin/rostopic
--- ros-1.6.8-orig/bin/rostopic	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rostopic	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rosversion ros-1.6.8/bin/rosversion
--- ros-1.6.8-orig/bin/rosversion	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rosversion	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 # Copyright (c) 2008, Willow Garage, Inc.
 # All rights reserved.
diff -Naur ros-1.6.8-orig/bin/roswtf ros-1.6.8/bin/roswtf
--- ros-1.6.8-orig/bin/roswtf	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/roswtf	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rxbag ros-1.6.8/bin/rxbag
--- ros-1.6.8-orig/bin/rxbag	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rxbag	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rxdeps ros-1.6.8/bin/rxdeps
--- ros-1.6.8-orig/bin/rxdeps	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rxdeps	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rxgraph ros-1.6.8/bin/rxgraph
--- ros-1.6.8-orig/bin/rxgraph	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rxgraph	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/bin/rxplot ros-1.6.8/bin/rxplot
--- ros-1.6.8-orig/bin/rxplot	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/bin/rxplot	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/core/rosbuild/bin/check_same_directories.py ros-1.6.8/core/rosbuild/bin/check_same_directories.py
--- ros-1.6.8-orig/core/rosbuild/bin/check_same_directories.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/rosbuild/bin/check_same_directories.py	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 # Simple script to check whether two directories are the same.  I'm doing
 # it in this script because the following command-line invocation produces
diff -Naur ros-1.6.8-orig/core/rosbuild/bin/download_checkmd5.py ros-1.6.8/core/rosbuild/bin/download_checkmd5.py
--- ros-1.6.8-orig/core/rosbuild/bin/download_checkmd5.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/rosbuild/bin/download_checkmd5.py	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 NAME="download_checkmd5.py"
 
diff -Naur ros-1.6.8-orig/core/rosbuild/bin/package_source.py ros-1.6.8/core/rosbuild/bin/package_source.py
--- ros-1.6.8-orig/core/rosbuild/bin/package_source.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/rosbuild/bin/package_source.py	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 """
 usage: %prog [args]
diff -Naur ros-1.6.8-orig/core/rosbuild/bin/roscheck ros-1.6.8/core/rosbuild/bin/roscheck
--- ros-1.6.8-orig/core/rosbuild/bin/roscheck	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/rosbuild/bin/roscheck	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 # Copyright (c) 2008, Willow Garage, Inc.
 # All rights reserved.
diff -Naur ros-1.6.8-orig/core/rosbuild/bin/rosgcov_summarize ros-1.6.8/core/rosbuild/bin/rosgcov_summarize
--- ros-1.6.8-orig/core/rosbuild/bin/rosgcov_summarize	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/rosbuild/bin/rosgcov_summarize	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import sys
 import fileinput
diff -Naur ros-1.6.8-orig/core/rosbuild/bin/rosmakeall_stackdist ros-1.6.8/core/rosbuild/bin/rosmakeall_stackdist
--- ros-1.6.8-orig/core/rosbuild/bin/rosmakeall_stackdist	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/rosbuild/bin/rosmakeall_stackdist	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import roslib
 import sys
diff -Naur ros-1.6.8-orig/core/rosbuild/bin/rostar ros-1.6.8/core/rosbuild/bin/rostar
--- ros-1.6.8-orig/core/rosbuild/bin/rostar	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/rosbuild/bin/rostar	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 """
 usage: %(progname)s [package] <ball.tar>
diff -Naur ros-1.6.8-orig/core/rosbuild/tests/count_cores.py ros-1.6.8/core/rosbuild/tests/count_cores.py
--- ros-1.6.8-orig/core/rosbuild/tests/count_cores.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/rosbuild/tests/count_cores.py	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import os
 print os.sysconf(os.sysconf_names['SC_NPROCESSORS_ONLN'])
diff -Naur ros-1.6.8-orig/core/roslib/scripts/gendeps ros-1.6.8/core/roslib/scripts/gendeps
--- ros-1.6.8-orig/core/roslib/scripts/gendeps	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/roslib/scripts/gendeps	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/core/roslib/src/roslib/gentools.py ros-1.6.8/core/roslib/src/roslib/gentools.py
--- ros-1.6.8-orig/core/roslib/src/roslib/gentools.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/roslib/src/roslib/gentools.py	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/core/roslib/src/roslib/launcher.py ros-1.6.8/core/roslib/src/roslib/launcher.py
--- ros-1.6.8-orig/core/roslib/src/roslib/launcher.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/roslib/src/roslib/launcher.py	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/core/roslib/src/roslib/manifestlib.py ros-1.6.8/core/roslib/src/roslib/manifestlib.py
--- ros-1.6.8-orig/core/roslib/src/roslib/manifestlib.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/roslib/src/roslib/manifestlib.py	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/core/roslib/src/roslib/manifest.py ros-1.6.8/core/roslib/src/roslib/manifest.py
--- ros-1.6.8-orig/core/roslib/src/roslib/manifest.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/roslib/src/roslib/manifest.py	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/core/roslib/src/roslib/os_detect.py ros-1.6.8/core/roslib/src/roslib/os_detect.py
--- ros-1.6.8-orig/core/roslib/src/roslib/os_detect.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/roslib/src/roslib/os_detect.py	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur ros-1.6.8-orig/core/roslib/src/roslib/stack_manifest.py ros-1.6.8/core/roslib/src/roslib/stack_manifest.py
--- ros-1.6.8-orig/core/roslib/src/roslib/stack_manifest.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/roslib/src/roslib/stack_manifest.py	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/core/roslib/src/roslib/stacks.py ros-1.6.8/core/roslib/src/roslib/stacks.py
--- ros-1.6.8-orig/core/roslib/src/roslib/stacks.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/core/roslib/src/roslib/stacks.py	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/test/test_roslib/test/fake_node.py ros-1.6.8/test/test_roslib/test/fake_node.py
--- ros-1.6.8-orig/test/test_roslib/test/fake_node.py	2012-02-03 17:32:20.000000000 +0100
+++ ros-1.6.8/test/test_roslib/test/fake_node.py	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # this node only exists to test find_node functionality
 
 print "hello"
diff -Naur ros-1.6.8-orig/test/test_rosmake/test/test_parallel_build.py ros-1.6.8/test/test_rosmake/test/test_parallel_build.py
--- ros-1.6.8-orig/test/test_rosmake/test/test_parallel_build.py	2012-02-03 17:32:20.000000000 +0100
+++ ros-1.6.8/test/test_rosmake/test/test_parallel_build.py	2012-04-19 23:50:44.317491478 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur ros-1.6.8-orig/tools/rosboost_cfg/src/rosboost_cfg/rosboost_cfg.py ros-1.6.8/tools/rosboost_cfg/src/rosboost_cfg/rosboost_cfg.py
--- ros-1.6.8-orig/tools/rosboost_cfg/src/rosboost_cfg/rosboost_cfg.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosboost_cfg/src/rosboost_cfg/rosboost_cfg.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2010, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/tools/roscreate/src/roscreate/core.py ros-1.6.8/tools/roscreate/src/roscreate/core.py
--- ros-1.6.8-orig/tools/roscreate/src/roscreate/core.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/roscreate/src/roscreate/core.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/tools/roscreate/src/roscreate/__init__.py ros-1.6.8/tools/roscreate/src/roscreate/__init__.py
--- ros-1.6.8-orig/tools/roscreate/src/roscreate/__init__.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/roscreate/src/roscreate/__init__.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/tools/roscreate/src/roscreate/roscreatepkg.py ros-1.6.8/tools/roscreate/src/roscreate/roscreatepkg.py
--- ros-1.6.8-orig/tools/roscreate/src/roscreate/roscreatepkg.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/roscreate/src/roscreate/roscreatepkg.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/tools/roscreate/src/roscreate/roscreatestack.py ros-1.6.8/tools/roscreate/src/roscreate/roscreatestack.py
--- ros-1.6.8-orig/tools/roscreate/src/roscreate/roscreatestack.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/roscreate/src/roscreate/roscreatestack.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/tools/rosdep/scripts/rosdep ros-1.6.8/tools/rosdep/scripts/rosdep
--- ros-1.6.8-orig/tools/rosdep/scripts/rosdep	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosdep/scripts/rosdep	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/tools/rosdep/src/rosdep/arch.py ros-1.6.8/tools/rosdep/src/rosdep/arch.py
--- ros-1.6.8-orig/tools/rosdep/src/rosdep/arch.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosdep/src/rosdep/arch.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
@@ -31,16 +31,20 @@
 from __future__ import with_statement
 from linux_helpers import *
 import os
-
+import roslib.os_detect
 import rosdep.base_rosdep
+import rosdep.installers
 
 ###### Arch SPECIALIZATION #########################
 
 def pacman_detect(p):
     return subprocess.call(['pacman', '-Q', p], stdout=subprocess.PIPE, stderr=subprocess.PIPE)    
 
-class Arch(rosdep.base_rosdep.RosdepBaseOS):
-
+class Arch(roslib.os_detect.Arch,rosdep.base_rosdep.RosdepBaseOS):
+    def __init__(self):
+        self.installers = {}
+        self.installers['pacman'] = rosdep.installers.PacmanInstaller
+        self.installers['default'] = rosdep.installers.PacmanInstaller
     def check_presence(self):
         filename = "/etc/arch-release"
         if os.path.exists(filename):
diff -Naur ros-1.6.8-orig/tools/rosdep/src/rosdep/base_rosdep.py ros-1.6.8/tools/rosdep/src/rosdep/base_rosdep.py
--- ros-1.6.8-orig/tools/rosdep/src/rosdep/base_rosdep.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosdep/src/rosdep/base_rosdep.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur ros-1.6.8-orig/tools/rosdep/src/rosdep/core.py ros-1.6.8/tools/rosdep/src/rosdep/core.py
--- ros-1.6.8-orig/tools/rosdep/src/rosdep/core.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosdep/src/rosdep/core.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur ros-1.6.8-orig/tools/rosdep/src/rosdep/cygwin.py ros-1.6.8/tools/rosdep/src/rosdep/cygwin.py
--- ros-1.6.8-orig/tools/rosdep/src/rosdep/cygwin.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosdep/src/rosdep/cygwin.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur ros-1.6.8-orig/tools/rosdep/src/rosdep/debian.py ros-1.6.8/tools/rosdep/src/rosdep/debian.py
--- ros-1.6.8-orig/tools/rosdep/src/rosdep/debian.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosdep/src/rosdep/debian.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur ros-1.6.8-orig/tools/rosdep/src/rosdep/freebsd.py ros-1.6.8/tools/rosdep/src/rosdep/freebsd.py
--- ros-1.6.8-orig/tools/rosdep/src/rosdep/freebsd.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosdep/src/rosdep/freebsd.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2010, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur ros-1.6.8-orig/tools/rosdep/src/rosdep/gentoo.py ros-1.6.8/tools/rosdep/src/rosdep/gentoo.py
--- ros-1.6.8-orig/tools/rosdep/src/rosdep/gentoo.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosdep/src/rosdep/gentoo.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur ros-1.6.8-orig/tools/rosdep/src/rosdep/installers.py ros-1.6.8/tools/rosdep/src/rosdep/installers.py
--- ros-1.6.8-orig/tools/rosdep/src/rosdep/installers.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosdep/src/rosdep/installers.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
@@ -594,4 +594,49 @@
                 ret_list.append(pkg)
         return ret_list
     
+class PacmanInstaller(InstallerAPI):
+    """
+    An implementation of the InstallerAPI for use on pacman systems
+    """
+    def __init__(self,arg_dict):
+        packages = arg_dict.get("packages","")
+        if type(packages) == type("string"):
+            packages = packages.split()
+
+        self.packages = packages
+        
+    def get_packages_to_install(self):
+         return list(set(self.packages) - set(self.pacman_detect(self.packages)))
+
+    def check_presence(self):
+        return len(self.get_packages_to_install()) == 0
 
+    def generate_package_install_command(self, default_yes = False, execute = True, display = True):
+        script = '!#/bin/bash\n#no script'
+        packages_to_install = self.get_packages_to_install()
+        if not packages_to_install:
+            script =  "#!/bin/bash\n#No Packages to install"
+        script = "#!/bin/bash\n#Packages %s\nsudo pacman -S "%packages_to_install + ' '.join(packages_to_install)        
+
+        if execute:
+            return rosdep.core.create_tempfile_from_string_and_execute(script)
+        elif display:
+            print("To install packages: %s would have executed script\n{{{\n%s\n}}}"%(packages_to_install, script))
+        return False
+
+    def pacman_detect(self, pkgs):
+        """ 
+        Given a list of package, return the list of installed packages.
+        """
+        ret_list = []
+        cmd = ['pacman','-Q']
+        cmd.extend(pkgs)
+        pop = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        (std_out, std_err) = pop.communicate()
+        pkg_list = std_out.split('\n')
+        for pkg in pkg_list:
+            pkg_row = pkg.split()
+            if len(pkg_row) == 2 and pkg_row[0] in pkgs:
+                ret_list.append( pkg_row[0])
+        return ret_list
+        
diff -Naur ros-1.6.8-orig/tools/rosdep/src/rosdep/main.py ros-1.6.8/tools/rosdep/src/rosdep/main.py
--- ros-1.6.8-orig/tools/rosdep/src/rosdep/main.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosdep/src/rosdep/main.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur ros-1.6.8-orig/tools/rosdep/src/rosdep/opensuse.py ros-1.6.8/tools/rosdep/src/rosdep/opensuse.py
--- ros-1.6.8-orig/tools/rosdep/src/rosdep/opensuse.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosdep/src/rosdep/opensuse.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur ros-1.6.8-orig/tools/rosdep/src/rosdep/osx.py ros-1.6.8/tools/rosdep/src/rosdep/osx.py
--- ros-1.6.8-orig/tools/rosdep/src/rosdep/osx.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosdep/src/rosdep/osx.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur ros-1.6.8-orig/tools/rosdep/src/rosdep/redhat.py ros-1.6.8/tools/rosdep/src/rosdep/redhat.py
--- ros-1.6.8-orig/tools/rosdep/src/rosdep/redhat.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosdep/src/rosdep/redhat.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur ros-1.6.8-orig/tools/rosmake/src/rosmake/engine.py ros-1.6.8/tools/rosmake/src/rosmake/engine.py
--- ros-1.6.8-orig/tools/rosmake/src/rosmake/engine.py	2012-02-03 17:32:20.000000000 +0100
+++ ros-1.6.8/tools/rosmake/src/rosmake/engine.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
diff -Naur ros-1.6.8-orig/tools/rosmake/src/rosmake/package_stats.py ros-1.6.8/tools/rosmake/src/rosmake/package_stats.py
--- ros-1.6.8-orig/tools/rosmake/src/rosmake/package_stats.py	2012-02-03 17:32:20.000000000 +0100
+++ ros-1.6.8/tools/rosmake/src/rosmake/package_stats.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
diff -Naur ros-1.6.8-orig/tools/rosmake/src/rosmake/parallel_build.py ros-1.6.8/tools/rosmake/src/rosmake/parallel_build.py
--- ros-1.6.8-orig/tools/rosmake/src/rosmake/parallel_build.py	2012-02-03 17:32:20.000000000 +0100
+++ ros-1.6.8/tools/rosmake/src/rosmake/parallel_build.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
diff -Naur ros-1.6.8-orig/tools/rospack/markstack ros-1.6.8/tools/rospack/markstack
--- ros-1.6.8-orig/tools/rospack/markstack	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rospack/markstack	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
diff -Naur ros-1.6.8-orig/tools/rospack/rosalldeps ros-1.6.8/tools/rospack/rosalldeps
--- ros-1.6.8-orig/tools/rospack/rosalldeps	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rospack/rosalldeps	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import os, sys
 from optparse import OptionParser
diff -Naur ros-1.6.8-orig/tools/rospack/rosallpkgs ros-1.6.8/tools/rospack/rosallpkgs
--- ros-1.6.8-orig/tools/rospack/rosallpkgs	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rospack/rosallpkgs	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 USAGE = 'USAGE: rosallpkgs [pkg1 pkg2...]'
 
diff -Naur ros-1.6.8-orig/tools/rosunit/bin/rosunit ros-1.6.8/tools/rosunit/bin/rosunit
--- ros-1.6.8-orig/tools/rosunit/bin/rosunit	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosunit/bin/rosunit	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2010, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/tools/rosunit/scripts/check_test_ran.py ros-1.6.8/tools/rosunit/scripts/check_test_ran.py
--- ros-1.6.8-orig/tools/rosunit/scripts/check_test_ran.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosunit/scripts/check_test_ran.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/tools/rosunit/scripts/clean_junit_xml.py ros-1.6.8/tools/rosunit/scripts/clean_junit_xml.py
--- ros-1.6.8-orig/tools/rosunit/scripts/clean_junit_xml.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosunit/scripts/clean_junit_xml.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/tools/rosunit/scripts/pycoverage_to_html.py ros-1.6.8/tools/rosunit/scripts/pycoverage_to_html.py
--- ros-1.6.8-orig/tools/rosunit/scripts/pycoverage_to_html.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosunit/scripts/pycoverage_to_html.py	2012-04-19 23:50:44.320824811 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/tools/rosunit/scripts/summarize_results.py ros-1.6.8/tools/rosunit/scripts/summarize_results.py
--- ros-1.6.8-orig/tools/rosunit/scripts/summarize_results.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosunit/scripts/summarize_results.py	2012-04-19 23:50:44.324158145 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/tools/rosunit/scripts/test_results_dir.py ros-1.6.8/tools/rosunit/scripts/test_results_dir.py
--- ros-1.6.8-orig/tools/rosunit/scripts/test_results_dir.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosunit/scripts/test_results_dir.py	2012-04-19 23:50:44.324158145 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/tools/rosunit/src/rosunit/junitxml.py ros-1.6.8/tools/rosunit/src/rosunit/junitxml.py
--- ros-1.6.8-orig/tools/rosunit/src/rosunit/junitxml.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosunit/src/rosunit/junitxml.py	2012-04-19 23:50:44.324158145 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros-1.6.8-orig/tools/rosunit/test/test_junitxml.py ros-1.6.8/tools/rosunit/test/test_junitxml.py
--- ros-1.6.8-orig/tools/rosunit/test/test_junitxml.py	2012-02-03 17:32:21.000000000 +0100
+++ ros-1.6.8/tools/rosunit/test/test_junitxml.py	2012-04-19 23:50:44.324158145 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
