diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7a277b5..9f3ff6a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -45,7 +45,7 @@ add_dependencies(nodeletlib ${PROJECT_NAME}_gencpp)
 add_executable(nodelet src/nodelet.cpp)
 target_link_libraries(nodelet ${BOOST_LIBRARIES}
                               ${catkin_LIBRARIES}
-                              nodeletlib
+                              nodeletlib uuid
 )
 
 # See: https://code.ros.org/trac/ros-pkg/ticket/5144
