diff -Naur pr2_object_manipulation-0.5.18-orig/applications/manipulation_worlds/meshes/make_ply.py pr2_object_manipulation/applications/manipulation_worlds/meshes/make_ply.py
--- pr2_object_manipulation-0.5.18-orig/applications/manipulation_worlds/meshes/make_ply.py	2012-04-28 00:18:51.000000000 +0200
+++ pr2_object_manipulation/applications/manipulation_worlds/meshes/make_ply.py	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 #write a ply file for a list of vertices and faces
 def write_ply(vertices, faces, outfile):
diff -Naur pr2_object_manipulation-0.5.18-orig/applications/manipulation_worlds/meshes/make_small_white_shelf.py pr2_object_manipulation/applications/manipulation_worlds/meshes/make_small_white_shelf.py
--- pr2_object_manipulation-0.5.18-orig/applications/manipulation_worlds/meshes/make_small_white_shelf.py	2012-04-28 00:18:51.000000000 +0200
+++ pr2_object_manipulation/applications/manipulation_worlds/meshes/make_small_white_shelf.py	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 from make_ply import *
 
 outfile = file("small_white_shelf.ply", 'w')
diff -Naur pr2_object_manipulation-0.5.18-orig/applications/manipulation_worlds/meshes/make_sushi_shelf.py pr2_object_manipulation/applications/manipulation_worlds/meshes/make_sushi_shelf.py
--- pr2_object_manipulation-0.5.18-orig/applications/manipulation_worlds/meshes/make_sushi_shelf.py	2012-04-28 00:18:51.000000000 +0200
+++ pr2_object_manipulation/applications/manipulation_worlds/meshes/make_sushi_shelf.py	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 from make_ply import *
 
 outfile = file("sushi_shelf.ply", 'w')
diff -Naur pr2_object_manipulation-0.5.18-orig/applications/pr2_interactive_manipulation/cfg/pickup_config.cfg pr2_object_manipulation/applications/pr2_interactive_manipulation/cfg/pickup_config.cfg
--- pr2_object_manipulation-0.5.18-orig/applications/pr2_interactive_manipulation/cfg/pickup_config.cfg	2012-04-28 00:18:51.000000000 +0200
+++ pr2_object_manipulation/applications/pr2_interactive_manipulation/cfg/pickup_config.cfg	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PACKAGE='pr2_interactive_manipulation'
 import roslib; roslib.load_manifest(PACKAGE)
diff -Naur pr2_object_manipulation-0.5.18-orig/manipulation/pr2_grasp_adjust/cfg/Debug.cfg pr2_object_manipulation/manipulation/pr2_grasp_adjust/cfg/Debug.cfg
--- pr2_object_manipulation-0.5.18-orig/manipulation/pr2_grasp_adjust/cfg/Debug.cfg	2012-04-28 00:18:51.000000000 +0200
+++ pr2_object_manipulation/manipulation/pr2_grasp_adjust/cfg/Debug.cfg	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 # Software License Agreement (BSD License)
 #
diff -Naur pr2_object_manipulation-0.5.18-orig/manipulation/pr2_grasp_adjust/cfg/Estimate.cfg pr2_object_manipulation/manipulation/pr2_grasp_adjust/cfg/Estimate.cfg
--- pr2_object_manipulation-0.5.18-orig/manipulation/pr2_grasp_adjust/cfg/Estimate.cfg	2012-04-28 00:18:51.000000000 +0200
+++ pr2_object_manipulation/manipulation/pr2_grasp_adjust/cfg/Estimate.cfg	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 # Software License Agreement (BSD License)
 #
diff -Naur pr2_object_manipulation-0.5.18-orig/manipulation/pr2_grasp_adjust/scripts/gripper_stereo_reactive_grasp.py pr2_object_manipulation/manipulation/pr2_grasp_adjust/scripts/gripper_stereo_reactive_grasp.py
--- pr2_object_manipulation-0.5.18-orig/manipulation/pr2_grasp_adjust/scripts/gripper_stereo_reactive_grasp.py	2012-04-28 00:18:51.000000000 +0200
+++ pr2_object_manipulation/manipulation/pr2_grasp_adjust/scripts/gripper_stereo_reactive_grasp.py	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur pr2_object_manipulation-0.5.18-orig/manipulation/pr2_gripper_reactive_approach/src/pr2_gripper_reactive_approach/hand_sensor_listeners.py pr2_object_manipulation/manipulation/pr2_gripper_reactive_approach/src/pr2_gripper_reactive_approach/hand_sensor_listeners.py
--- pr2_object_manipulation-0.5.18-orig/manipulation/pr2_gripper_reactive_approach/src/pr2_gripper_reactive_approach/hand_sensor_listeners.py	2012-04-28 00:18:51.000000000 +0200
+++ pr2_object_manipulation/manipulation/pr2_gripper_reactive_approach/src/pr2_gripper_reactive_approach/hand_sensor_listeners.py	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur pr2_object_manipulation-0.5.18-orig/manipulation/pr2_gripper_reactive_approach/src/pr2_gripper_reactive_approach/reactive_grasp.py pr2_object_manipulation/manipulation/pr2_gripper_reactive_approach/src/pr2_gripper_reactive_approach/reactive_grasp.py
--- pr2_object_manipulation-0.5.18-orig/manipulation/pr2_gripper_reactive_approach/src/pr2_gripper_reactive_approach/reactive_grasp.py	2012-04-28 00:18:51.000000000 +0200
+++ pr2_object_manipulation/manipulation/pr2_gripper_reactive_approach/src/pr2_gripper_reactive_approach/reactive_grasp.py	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur pr2_object_manipulation-0.5.18-orig/manipulation/pr2_handy_tools/scripts/point_head_test.py pr2_object_manipulation/manipulation/pr2_handy_tools/scripts/point_head_test.py
--- pr2_object_manipulation-0.5.18-orig/manipulation/pr2_handy_tools/scripts/point_head_test.py	2012-04-28 00:18:51.000000000 +0200
+++ pr2_object_manipulation/manipulation/pr2_handy_tools/scripts/point_head_test.py	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 import roslib; roslib.load_manifest('busbots_unloader')
 import rospy
diff -Naur pr2_object_manipulation-0.5.18-orig/manipulation/pr2_manipulation_controllers/scripts/posture.py pr2_object_manipulation/manipulation/pr2_manipulation_controllers/scripts/posture.py
--- pr2_object_manipulation-0.5.18-orig/manipulation/pr2_manipulation_controllers/scripts/posture.py	2012-04-28 00:18:51.000000000 +0200
+++ pr2_object_manipulation/manipulation/pr2_manipulation_controllers/scripts/posture.py	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur pr2_object_manipulation-0.5.18-orig/manipulation/pr2_object_manipulation_launch/scripts/change_joint_controller_gains.py pr2_object_manipulation/manipulation/pr2_object_manipulation_launch/scripts/change_joint_controller_gains.py
--- pr2_object_manipulation-0.5.18-orig/manipulation/pr2_object_manipulation_launch/scripts/change_joint_controller_gains.py	2012-04-28 00:18:51.000000000 +0200
+++ pr2_object_manipulation/manipulation/pr2_object_manipulation_launch/scripts/change_joint_controller_gains.py	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 import roslib; roslib.load_manifest('pr2_object_manipulation_launch')
 import rospy
diff -Naur pr2_object_manipulation-0.5.18-orig/manipulation/pr2_object_manipulation_launch/scripts/kinect_monitor.py pr2_object_manipulation/manipulation/pr2_object_manipulation_launch/scripts/kinect_monitor.py
--- pr2_object_manipulation-0.5.18-orig/manipulation/pr2_object_manipulation_launch/scripts/kinect_monitor.py	2012-04-28 00:18:51.000000000 +0200
+++ pr2_object_manipulation/manipulation/pr2_object_manipulation_launch/scripts/kinect_monitor.py	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import roslib; roslib.load_manifest('sensor_msgs')
 
diff -Naur pr2_object_manipulation-0.5.18-orig/perception/vfh_recognition/nodes/generate_views_vfh.py pr2_object_manipulation/perception/vfh_recognition/nodes/generate_views_vfh.py
--- pr2_object_manipulation-0.5.18-orig/perception/vfh_recognition/nodes/generate_views_vfh.py	2012-04-28 00:18:52.000000000 +0200
+++ pr2_object_manipulation/perception/vfh_recognition/nodes/generate_views_vfh.py	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 #
 # Software License Agreement (BSD License)
diff -Naur pr2_object_manipulation-0.5.18-orig/perception/vfh_recognition/nodes/views_generator/3Dscan.py pr2_object_manipulation/perception/vfh_recognition/nodes/views_generator/3Dscan.py
--- pr2_object_manipulation-0.5.18-orig/perception/vfh_recognition/nodes/views_generator/3Dscan.py	2012-04-28 00:18:52.000000000 +0200
+++ pr2_object_manipulation/perception/vfh_recognition/nodes/views_generator/3Dscan.py	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import os
 import time
diff -Naur pr2_object_manipulation-0.5.18-orig/perception/vfh_recognition/nodes/views_generator/views_generator.py pr2_object_manipulation/perception/vfh_recognition/nodes/views_generator/views_generator.py
--- pr2_object_manipulation-0.5.18-orig/perception/vfh_recognition/nodes/views_generator/views_generator.py	2012-04-28 00:18:52.000000000 +0200
+++ pr2_object_manipulation/perception/vfh_recognition/nodes/views_generator/views_generator.py	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 import os
 import time
diff -Naur pr2_object_manipulation-0.5.18-orig/perception/vfh_recognizer_db/nodes/generate_views_vfh.py pr2_object_manipulation/perception/vfh_recognizer_db/nodes/generate_views_vfh.py
--- pr2_object_manipulation-0.5.18-orig/perception/vfh_recognizer_db/nodes/generate_views_vfh.py	2012-04-28 00:18:52.000000000 +0200
+++ pr2_object_manipulation/perception/vfh_recognizer_db/nodes/generate_views_vfh.py	2012-05-08 16:45:40.283903201 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 #
 # Software License Agreement (BSD License)
diff -Naur pr2_object_manipulation-0.5.18-orig/perception/vfh_recognizer_db/src/vfh_recognition.cpp pr2_object_manipulation/perception/vfh_recognizer_db/src/vfh_recognition.cpp
--- pr2_object_manipulation-0.5.18-orig/perception/vfh_recognizer_db/src/vfh_recognition.cpp	2012-04-28 00:18:52.000000000 +0200
+++ pr2_object_manipulation/perception/vfh_recognizer_db/src/vfh_recognition.cpp	2012-05-08 16:45:40.283903201 +0200
@@ -322,14 +322,14 @@
         {
           //SAVE the kdtree + training_list to the database
           flann::save_to_file (data_, training_data_h5_file_name.string ().c_str (), "training_data");
-          saveFileList (models_, training_data_list_file_name.string ().c_str ());
+          this->saveFileList (models_, training_data_list_file_name.string ().c_str ());
           index_->save (index_filename.string ().c_str ());
         }
       }
     }
     else
     {
-      loadFileList (models_, training_data_list_file_name.string ());
+      this->loadFileList (models_, training_data_list_file_name.string ());
       flann::load_from_file (data_, training_data_h5_file_name.string (), "training_data");
       index_ = new flann::Index<DistT> (data_, flann::SavedIndexParams (index_filename.string ().c_str ()));
       index_->buildIndex ();
diff -Naur pr2_object_manipulation-0.5.18-orig/perception/vfh_recognizer_fs/nodes/generate_views_vfh.py pr2_object_manipulation/perception/vfh_recognizer_fs/nodes/generate_views_vfh.py
--- pr2_object_manipulation-0.5.18-orig/perception/vfh_recognizer_fs/nodes/generate_views_vfh.py	2012-04-28 00:18:52.000000000 +0200
+++ pr2_object_manipulation/perception/vfh_recognizer_fs/nodes/generate_views_vfh.py	2012-05-08 16:46:11.980571065 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 #
 # Software License Agreement (BSD License)
diff -Naur pr2_object_manipulation-0.5.18-orig/perception/vfh_recognizer_fs/src/vfh_recognition.cpp pr2_object_manipulation/perception/vfh_recognizer_fs/src/vfh_recognition.cpp
--- pr2_object_manipulation-0.5.18-orig/perception/vfh_recognizer_fs/src/vfh_recognition.cpp	2012-04-28 00:18:52.000000000 +0200
+++ pr2_object_manipulation/perception/vfh_recognizer_fs/src/vfh_recognition.cpp	2012-05-08 16:46:11.990571064 +0200
@@ -211,13 +211,13 @@
         {
           //SAVE the kdtree + training_list
           flann::save_to_file (data_, training_data_h5_file_name.string ().c_str (), "training_data");
-          saveFileList (models_, training_data_list_file_name.string ().c_str ());
+          this->saveFileList (models_, training_data_list_file_name.string ().c_str ());
           index_->save (index_filename.string ().c_str ());
         }
       }
       else
       {
-        loadFileList (models_, training_data_list_file_name.string ());
+        this->loadFileList (models_, training_data_list_file_name.string ());
         flann::load_from_file (data_, training_data_h5_file_name.string (), "training_data");
         std::cout << "Training data found. Loaded" << (int)data_.rows << "VFH models from:" <<
                          training_data_h5_file_name.string () << training_data_list_file_name.string () << std::endl;
