diff -Naur common-1.6.1/actionlib/src/actionlib/action_client.py common/actionlib/src/actionlib/action_client.py
--- common-1.6.1/actionlib/src/actionlib/action_client.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/src/actionlib/action_client.py	2012-04-19 00:13:27.361480124 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur common-1.6.1/actionlib/src/actionlib/action_server.py common/actionlib/src/actionlib/action_server.py
--- common-1.6.1/actionlib/src/actionlib/action_server.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/src/actionlib/action_server.py	2012-04-19 00:13:27.351480124 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur common-1.6.1/actionlib/src/actionlib/exceptions.py common/actionlib/src/actionlib/exceptions.py
--- common-1.6.1/actionlib/src/actionlib/exceptions.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/src/actionlib/exceptions.py	2012-04-19 00:13:27.371480125 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2010, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur common-1.6.1/actionlib/src/actionlib/goal_id_generator.py common/actionlib/src/actionlib/goal_id_generator.py
--- common-1.6.1/actionlib/src/actionlib/goal_id_generator.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/src/actionlib/goal_id_generator.py	2012-04-19 00:13:27.354813457 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur common-1.6.1/actionlib/src/actionlib/simple_action_client.py common/actionlib/src/actionlib/simple_action_client.py
--- common-1.6.1/actionlib/src/actionlib/simple_action_client.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/src/actionlib/simple_action_client.py	2012-04-19 00:13:27.364813457 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur common-1.6.1/actionlib/src/actionlib/simple_action_server.py common/actionlib/src/actionlib/simple_action_server.py
--- common-1.6.1/actionlib/src/actionlib/simple_action_server.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/src/actionlib/simple_action_server.py	2012-04-19 00:13:27.368146791 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur common-1.6.1/actionlib/test/exercise_simple_client.py common/actionlib/test/exercise_simple_client.py
--- common-1.6.1/actionlib/test/exercise_simple_client.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/test/exercise_simple_client.py	2012-04-19 00:13:27.338146790 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur common-1.6.1/actionlib/test/mock_simple_server.py common/actionlib/test/mock_simple_server.py
--- common-1.6.1/actionlib/test/mock_simple_server.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/test/mock_simple_server.py	2012-04-19 00:13:27.334813456 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur common-1.6.1/actionlib/test/ref_server.py common/actionlib/test/ref_server.py
--- common-1.6.1/actionlib/test/ref_server.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/test/ref_server.py	2012-04-19 00:13:27.341480124 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur common-1.6.1/actionlib/test/ref_simple_server.py common/actionlib/test/ref_simple_server.py
--- common-1.6.1/actionlib/test/ref_simple_server.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/test/ref_simple_server.py	2012-04-19 00:13:27.321480122 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur common-1.6.1/actionlib/test/simple_python_client_test.py common/actionlib/test/simple_python_client_test.py
--- common-1.6.1/actionlib/test/simple_python_client_test.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/test/simple_python_client_test.py	2012-04-19 00:13:27.348146790 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur common-1.6.1/actionlib/test/test_imports.py common/actionlib/test/test_imports.py
--- common-1.6.1/actionlib/test/test_imports.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/test/test_imports.py	2012-04-19 00:13:27.331480123 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur common-1.6.1/actionlib/test/test_ref_action_server.py common/actionlib/test/test_ref_action_server.py
--- common-1.6.1/actionlib/test/test_ref_action_server.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/test/test_ref_action_server.py	2012-04-19 00:13:27.344813457 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur common-1.6.1/actionlib/test/test_ref_simple_action_server.py common/actionlib/test/test_ref_simple_action_server.py
--- common-1.6.1/actionlib/test/test_ref_simple_action_server.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/test/test_ref_simple_action_server.py	2012-04-19 00:13:27.328146790 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur common-1.6.1/actionlib/test/test_server_components.py common/actionlib/test/test_server_components.py
--- common-1.6.1/actionlib/test/test_server_components.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/test/test_server_components.py	2012-04-19 00:13:27.324813456 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur common-1.6.1/actionlib/tools/axserver.py common/actionlib/tools/axserver.py
--- common-1.6.1/actionlib/tools/axserver.py	2011-10-20 01:57:16.000000000 +0200
+++ common/actionlib/tools/axserver.py	2012-04-19 00:13:27.318146789 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
