diff -Naur ros_realtime-0.5.3/rosrt/test/test_publisher_no_initialize.py ros_realtime/rosrt/test/test_publisher_no_initialize.py
--- ros_realtime-0.5.3/rosrt/test/test_publisher_no_initialize.py	2011-10-20 01:51:59.000000000 +0200
+++ ros_realtime/rosrt/test/test_publisher_no_initialize.py	2012-05-08 14:59:40.463663413 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
