diff -Naur common_msgs-1.6.0/actionlib_msgs/genaction.py common_msgs/actionlib_msgs/genaction.py
--- common_msgs-1.6.0/actionlib_msgs/genaction.py	2011-08-08 01:36:14.000000000 +0200
+++ common_msgs/actionlib_msgs/genaction.py	2012-04-15 23:52:26.704547494 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur common_msgs-1.6.0/actionlib_msgs/genaction-rosbuild2.py common_msgs/actionlib_msgs/genaction-rosbuild2.py
--- common_msgs-1.6.0/actionlib_msgs/genaction-rosbuild2.py	2011-08-08 01:36:14.000000000 +0200
+++ common_msgs/actionlib_msgs/genaction-rosbuild2.py	2012-04-15 23:52:26.701214164 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
diff -Naur common_msgs-1.6.0/stereo_msgs/migration_rules/split_rawstereo.py common_msgs/stereo_msgs/migration_rules/split_rawstereo.py
--- common_msgs-1.6.0/stereo_msgs/migration_rules/split_rawstereo.py	2011-08-08 01:36:14.000000000 +0200
+++ common_msgs/stereo_msgs/migration_rules/split_rawstereo.py	2012-04-15 23:52:26.697880832 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PKG = 'stereo_msgs'
 import roslib; roslib.load_manifest(PKG)
diff -Naur common_msgs-1.6.0/test_common_msgs/test/test_common_msgs_migration.py common_msgs/test_common_msgs/test/test_common_msgs_migration.py
--- common_msgs-1.6.0/test_common_msgs/test/test_common_msgs_migration.py	2011-08-08 01:36:14.000000000 +0200
+++ common_msgs/test_common_msgs/test/test_common_msgs_migration.py	2012-04-15 23:52:26.711214154 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
