diff -Naur pr2_controllers-1.2.6/control_toolbox/scripts/tune_pid.py pr2_controllers/control_toolbox/scripts/tune_pid.py
--- pr2_controllers-1.2.6/control_toolbox/scripts/tune_pid.py	2011-01-04 23:53:20.000000000 +0100
+++ pr2_controllers/control_toolbox/scripts/tune_pid.py	2012-04-29 17:39:05.351664858 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur pr2_controllers-1.2.6/pr2_mechanism_controllers/include/pr2_mechanism_controllers/pr2_odometry.h pr2_controllers/pr2_mechanism_controllers/include/pr2_mechanism_controllers/pr2_odometry.h
--- pr2_controllers-1.2.6/pr2_mechanism_controllers/include/pr2_mechanism_controllers/pr2_odometry.h	2011-01-04 23:53:20.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/include/pr2_mechanism_controllers/pr2_odometry.h	2012-04-29 17:39:05.438331527 +0200
@@ -32,7 +32,7 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  *********************************************************************/
 
-#include <Eigen/Array>
+#include <Eigen/Core>
 #include <Eigen/SVD>
 #include <nav_msgs/Odometry.h>
 #include <pr2_mechanism_controllers/Odometer.h>
diff -Naur pr2_controllers-1.2.6/pr2_mechanism_controllers/scripts/control_base_position_ground_truth.py pr2_controllers/pr2_mechanism_controllers/scripts/control_base_position_ground_truth.py
--- pr2_controllers-1.2.6/pr2_mechanism_controllers/scripts/control_base_position_ground_truth.py	2011-01-04 23:53:20.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/scripts/control_base_position_ground_truth.py	2012-04-29 17:39:05.438331527 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2008, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur pr2_controllers-1.2.6/pr2_mechanism_controllers/scripts/control_base_position.py pr2_controllers/pr2_mechanism_controllers/scripts/control_base_position.py
--- pr2_controllers-1.2.6/pr2_mechanism_controllers/scripts/control_base_position.py	2011-01-04 23:53:20.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/scripts/control_base_position.py	2012-04-29 17:39:05.438331527 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2008, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur pr2_controllers-1.2.6/pr2_mechanism_controllers/scripts/control_laser.py pr2_controllers/pr2_mechanism_controllers/scripts/control_laser.py
--- pr2_controllers-1.2.6/pr2_mechanism_controllers/scripts/control_laser.py	2011-01-04 23:53:20.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/scripts/control_laser.py	2012-04-29 17:39:05.438331527 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PKG = "pr2_mechanism_controllers"
 
diff -Naur pr2_controllers-1.2.6/pr2_mechanism_controllers/scripts/pointhead.py pr2_controllers/pr2_mechanism_controllers/scripts/pointhead.py
--- pr2_controllers-1.2.6/pr2_mechanism_controllers/scripts/pointhead.py	2011-01-04 23:53:20.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/scripts/pointhead.py	2012-04-29 17:39:05.438331527 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2008, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur pr2_controllers-1.2.6/pr2_mechanism_controllers/scripts/send_laser_traj_cmd_ms2.py pr2_controllers/pr2_mechanism_controllers/scripts/send_laser_traj_cmd_ms2.py
--- pr2_controllers-1.2.6/pr2_mechanism_controllers/scripts/send_laser_traj_cmd_ms2.py	2011-01-04 23:53:20.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/scripts/send_laser_traj_cmd_ms2.py	2012-04-29 17:39:05.438331527 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PKG = "pr2_mechanism_controllers"
 
diff -Naur pr2_controllers-1.2.6/pr2_mechanism_controllers/scripts/send_periodic_cmd.py pr2_controllers/pr2_mechanism_controllers/scripts/send_periodic_cmd.py
--- pr2_controllers-1.2.6/pr2_mechanism_controllers/scripts/send_periodic_cmd.py	2011-01-04 23:53:20.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/scripts/send_periodic_cmd.py	2012-04-29 17:39:05.438331527 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PKG = "pr2_mechanism_controllers"
 
diff -Naur pr2_controllers-1.2.6/pr2_mechanism_controllers/scripts/send_periodic_cmd_srv.py pr2_controllers/pr2_mechanism_controllers/scripts/send_periodic_cmd_srv.py
--- pr2_controllers-1.2.6/pr2_mechanism_controllers/scripts/send_periodic_cmd_srv.py	2011-01-04 23:53:20.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/scripts/send_periodic_cmd_srv.py	2012-04-29 17:39:05.438331527 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PKG = "pr2_mechanism_controllers"
 
diff -Naur pr2_controllers-1.2.6/pr2_mechanism_controllers/src/pr2_odometry.cpp pr2_controllers/pr2_mechanism_controllers/src/pr2_odometry.cpp
--- pr2_controllers-1.2.6/pr2_mechanism_controllers/src/pr2_odometry.cpp	2011-01-04 23:53:20.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/src/pr2_odometry.cpp	2012-04-29 17:39:05.438331527 +0200
@@ -389,7 +389,7 @@
     cbv_soln_ = iterativeLeastSquares(cbv_lhs_, cbv_rhs_, ils_weight_type_, ils_max_iterations_);
 
     odometry_residual_ = cbv_lhs_ * cbv_soln_ - cbv_rhs_;
-    odometry_residual_max_ = odometry_residual_.cwise().abs().maxCoeff();
+    odometry_residual_max_ = odometry_residual_.array().abs().maxCoeff();
     ROS_DEBUG("Odometry:: base velocity: %f, %f, %f",cbv_soln_(0,0), cbv_soln_(1,0), cbv_soln_(2,0));
     ROS_DEBUG("Odometry:: odometry residual: %f",odometry_residual_max_);
     odom_vel_.linear.x = cbv_soln_(0, 0);
@@ -423,8 +423,8 @@
 
         if(verbose_)
           tmp_start = ros::Time::now();
-        Eigen::SVD<Eigen::MatrixXf> svdOfFit(fit_lhs_);
-        svdOfFit.solve(fit_rhs_, &fit_soln_);
+        Eigen::JacobiSVD<Eigen::MatrixXf> svdOfFit(fit_lhs_);
+        fit_soln_ = svdOfFit.solve(fit_rhs_);
 
         ROS_DEBUG("Odometry:: fit_soln_: %f %f %f",fit_soln_(0,0), fit_soln_(1,0), fit_soln_(2,0));
 
@@ -451,7 +451,7 @@
           }
 
         fit_residual_ = rhs - lhs * fit_soln_;
-        if(odometry_residual_.cwise().abs().maxCoeff() <= ODOMETRY_THRESHOLD)
+        if(odometry_residual_.array().abs().maxCoeff() <= ODOMETRY_THRESHOLD)
         {
           ROS_DEBUG("Breaking out since odometry looks good");
           break;
