diff -Naur pr2_controllers-1.6.2/control_toolbox/scripts/tune_pid.py pr2_controllers/control_toolbox/scripts/tune_pid.py
--- pr2_controllers-1.6.2/control_toolbox/scripts/tune_pid.py	2012-02-07 03:27:09.000000000 +0100
+++ pr2_controllers/control_toolbox/scripts/tune_pid.py	2012-04-29 17:46:42.628339604 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur pr2_controllers-1.6.2/pr2_mechanism_controllers/scripts/control_base_position_ground_truth.py pr2_controllers/pr2_mechanism_controllers/scripts/control_base_position_ground_truth.py
--- pr2_controllers-1.6.2/pr2_mechanism_controllers/scripts/control_base_position_ground_truth.py	2012-02-07 03:27:09.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/scripts/control_base_position_ground_truth.py	2012-04-29 17:46:42.645006270 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2008, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur pr2_controllers-1.6.2/pr2_mechanism_controllers/scripts/control_base_position.py pr2_controllers/pr2_mechanism_controllers/scripts/control_base_position.py
--- pr2_controllers-1.6.2/pr2_mechanism_controllers/scripts/control_base_position.py	2012-02-07 03:27:09.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/scripts/control_base_position.py	2012-04-29 17:46:42.648339604 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2008, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur pr2_controllers-1.6.2/pr2_mechanism_controllers/scripts/control_laser.py pr2_controllers/pr2_mechanism_controllers/scripts/control_laser.py
--- pr2_controllers-1.6.2/pr2_mechanism_controllers/scripts/control_laser.py	2012-02-07 03:27:09.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/scripts/control_laser.py	2012-04-29 17:46:42.638339604 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PKG = "pr2_mechanism_controllers"
 
diff -Naur pr2_controllers-1.6.2/pr2_mechanism_controllers/scripts/pointhead.py pr2_controllers/pr2_mechanism_controllers/scripts/pointhead.py
--- pr2_controllers-1.6.2/pr2_mechanism_controllers/scripts/pointhead.py	2012-02-07 03:27:09.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/scripts/pointhead.py	2012-04-29 17:46:42.651672938 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2008, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur pr2_controllers-1.6.2/pr2_mechanism_controllers/scripts/send_laser_traj_cmd_ms2.py pr2_controllers/pr2_mechanism_controllers/scripts/send_laser_traj_cmd_ms2.py
--- pr2_controllers-1.6.2/pr2_mechanism_controllers/scripts/send_laser_traj_cmd_ms2.py	2012-02-07 03:27:09.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/scripts/send_laser_traj_cmd_ms2.py	2012-04-29 17:46:42.631672936 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PKG = "pr2_mechanism_controllers"
 
diff -Naur pr2_controllers-1.6.2/pr2_mechanism_controllers/scripts/send_periodic_cmd.py pr2_controllers/pr2_mechanism_controllers/scripts/send_periodic_cmd.py
--- pr2_controllers-1.6.2/pr2_mechanism_controllers/scripts/send_periodic_cmd.py	2012-02-07 03:27:09.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/scripts/send_periodic_cmd.py	2012-04-29 17:46:42.641672937 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PKG = "pr2_mechanism_controllers"
 
diff -Naur pr2_controllers-1.6.2/pr2_mechanism_controllers/scripts/send_periodic_cmd_srv.py pr2_controllers/pr2_mechanism_controllers/scripts/send_periodic_cmd_srv.py
--- pr2_controllers-1.6.2/pr2_mechanism_controllers/scripts/send_periodic_cmd_srv.py	2012-02-07 03:27:09.000000000 +0100
+++ pr2_controllers/pr2_mechanism_controllers/scripts/send_periodic_cmd_srv.py	2012-04-29 17:46:42.635006270 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PKG = "pr2_mechanism_controllers"
 
diff -Naur pr2_controllers-1.6.2/robot_mechanism_controllers/posture.py pr2_controllers/robot_mechanism_controllers/posture.py
--- pr2_controllers-1.6.2/robot_mechanism_controllers/posture.py	2012-02-07 03:27:09.000000000 +0100
+++ pr2_controllers/robot_mechanism_controllers/posture.py	2012-04-29 17:46:42.655006271 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 # 
