diff -Naur turtlebot_apps-0.9.2/turtlebot_actions/test/fiducial_standalone_test/test_client.py turtlebot_apps/turtlebot_actions/test/fiducial_standalone_test/test_client.py
--- turtlebot_apps-0.9.2/turtlebot_actions/test/fiducial_standalone_test/test_client.py	2012-02-23 03:03:36.000000000 +0100
+++ turtlebot_apps/turtlebot_actions/test/fiducial_standalone_test/test_client.py	2012-05-02 17:34:43.083425744 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import roslib
 roslib.load_manifest('turtlebot_actions')
 
diff -Naur turtlebot_apps-0.9.2/turtlebot_follower/cfg/Follower.cfg turtlebot_apps/turtlebot_follower/cfg/Follower.cfg
--- turtlebot_apps-0.9.2/turtlebot_follower/cfg/Follower.cfg	2012-02-23 03:03:36.000000000 +0100
+++ turtlebot_apps/turtlebot_follower/cfg/Follower.cfg	2012-05-02 17:34:43.083425744 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python                                                                                          
+#! /usr/bin/env python2
 # Software License Agreement (BSD License)                                                                      
 #                                                                                                               
 # Copyright (c) 2009, Willow Garage, Inc.                                                                       
diff -Naur turtlebot_apps-0.9.2/turtlebot_teleop/bin/turtlebot_teleop_key turtlebot_apps/turtlebot_teleop/bin/turtlebot_teleop_key
--- turtlebot_apps-0.9.2/turtlebot_teleop/bin/turtlebot_teleop_key	2012-02-23 03:03:36.000000000 +0100
+++ turtlebot_apps/turtlebot_teleop/bin/turtlebot_teleop_key	2012-05-02 17:34:43.080092412 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 # Copyright (c) 2011, Willow Garage, Inc.
 # All rights reserved.
