diff -Naur bond_core-1.6.1/test_bond/scripts/tester.py bond_core/test_bond/scripts/tester.py
--- bond_core-1.6.1/test_bond/scripts/tester.py	2011-10-20 01:47:56.000000000 +0200
+++ bond_core/test_bond/scripts/tester.py	2012-04-17 20:00:56.411830509 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur bond_core-1.6.1/test_bond/test/exercise_bond.py bond_core/test_bond/test/exercise_bond.py
--- bond_core-1.6.1/test_bond/test/exercise_bond.py	2011-10-20 01:47:56.000000000 +0200
+++ bond_core/test_bond/test/exercise_bond.py	2012-04-17 20:00:56.418497176 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur bond_core-1.6.1/test_bond/test/test_callbacks_py.py bond_core/test_bond/test/test_callbacks_py.py
--- bond_core-1.6.1/test_bond/test/test_callbacks_py.py	2011-10-20 01:47:56.000000000 +0200
+++ bond_core/test_bond/test/test_callbacks_py.py	2012-04-17 20:00:56.415163843 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2009, Willow Garage, Inc.
 # All rights reserved.
 #
