diff -Naur ros_comm-1.6.6/clients/cpp/roscpp/scripts/genmsg_cpp.py ros_comm/clients/cpp/roscpp/scripts/genmsg_cpp.py
--- ros_comm-1.6.6/clients/cpp/roscpp/scripts/genmsg_cpp.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/clients/cpp/roscpp/scripts/genmsg_cpp.py	2012-04-15 22:54:42.144554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/clients/cpp/roscpp/scripts/gensrv_cpp.py ros_comm/clients/cpp/roscpp/scripts/gensrv_cpp.py
--- ros_comm-1.6.6/clients/cpp/roscpp/scripts/gensrv_cpp.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/clients/cpp/roscpp/scripts/gensrv_cpp.py	2012-04-15 22:54:42.141221463 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/clients/cpp/roscpp/scripts/rosbuild2/genmsg_cpp.py ros_comm/clients/cpp/roscpp/scripts/rosbuild2/genmsg_cpp.py
--- ros_comm-1.6.6/clients/cpp/roscpp/scripts/rosbuild2/genmsg_cpp.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/clients/cpp/roscpp/scripts/rosbuild2/genmsg_cpp.py	2012-04-15 22:54:42.157888130 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/clients/cpp/roscpp/scripts/rosbuild2/gensrv_cpp.py ros_comm/clients/cpp/roscpp/scripts/rosbuild2/gensrv_cpp.py
--- ros_comm-1.6.6/clients/cpp/roscpp/scripts/rosbuild2/gensrv_cpp.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/clients/cpp/roscpp/scripts/rosbuild2/gensrv_cpp.py	2012-04-15 22:54:42.147888130 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/clients/cpp/roscpp/scripts/rosbuild2/msg_gen.py ros_comm/clients/cpp/roscpp/scripts/rosbuild2/msg_gen.py
--- ros_comm-1.6.6/clients/cpp/roscpp/scripts/rosbuild2/msg_gen.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/clients/cpp/roscpp/scripts/rosbuild2/msg_gen.py	2012-04-15 22:54:42.151221463 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/clients/cpp/roscpp/src/roscpp/msg_gen.py ros_comm/clients/cpp/roscpp/src/roscpp/msg_gen.py
--- ros_comm-1.6.6/clients/cpp/roscpp/src/roscpp/msg_gen.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/clients/cpp/roscpp/src/roscpp/msg_gen.py	2012-04-15 22:54:42.161221463 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/clients/roslisp/scripts/genmsg_lisp.py ros_comm/clients/roslisp/scripts/genmsg_lisp.py
--- ros_comm-1.6.6/clients/roslisp/scripts/genmsg_lisp.py	2011-12-08 17:13:55.000000000 +0100
+++ ros_comm/clients/roslisp/scripts/genmsg_lisp.py	2012-04-15 22:54:42.137888130 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/clients/rospy/scripts/genmsg_py.py ros_comm/clients/rospy/scripts/genmsg_py.py
--- ros_comm-1.6.6/clients/rospy/scripts/genmsg_py.py	2011-12-08 17:13:55.000000000 +0100
+++ ros_comm/clients/rospy/scripts/genmsg_py.py	2012-04-15 22:54:42.164554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/clients/rospy/scripts/gensrv_py.py ros_comm/clients/rospy/scripts/gensrv_py.py
--- ros_comm-1.6.6/clients/rospy/scripts/gensrv_py.py	2011-12-08 17:13:55.000000000 +0100
+++ ros_comm/clients/rospy/scripts/gensrv_py.py	2012-04-15 22:54:42.167888130 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/clients/rospy/scripts/rosbuild2/genmsg_py.py ros_comm/clients/rospy/scripts/rosbuild2/genmsg_py.py
--- ros_comm-1.6.6/clients/rospy/scripts/rosbuild2/genmsg_py.py	2011-12-08 17:13:55.000000000 +0100
+++ ros_comm/clients/rospy/scripts/rosbuild2/genmsg_py.py	2012-04-15 22:54:42.171221463 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/clients/rospy/scripts/rosbuild2/gensrv_py.py ros_comm/clients/rospy/scripts/rosbuild2/gensrv_py.py
--- ros_comm-1.6.6/clients/rospy/scripts/rosbuild2/gensrv_py.py	2011-12-08 17:13:55.000000000 +0100
+++ ros_comm/clients/rospy/scripts/rosbuild2/gensrv_py.py	2012-04-15 22:54:42.174554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/nodes/add_two_ints_client ros_comm/test/test_ros/nodes/add_two_ints_client
--- ros_comm-1.6.6/test/test_ros/nodes/add_two_ints_client	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/nodes/add_two_ints_client	2012-04-15 22:54:41.794554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/nodes/add_two_ints_server ros_comm/test/test_ros/nodes/add_two_ints_server
--- ros_comm-1.6.6/test/test_ros/nodes/add_two_ints_server	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/nodes/add_two_ints_server	2012-04-15 22:54:41.781221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/nodes/fail_two_ints_server ros_comm/test/test_ros/nodes/fail_two_ints_server
--- ros_comm-1.6.6/test/test_ros/nodes/fail_two_ints_server	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/nodes/fail_two_ints_server	2012-04-15 22:54:41.801221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/nodes/fake_time.py ros_comm/test/test_ros/nodes/fake_time.py
--- ros_comm-1.6.6/test/test_ros/nodes/fake_time.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/nodes/fake_time.py	2012-04-15 22:54:41.791221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2010, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/nodes/listener.py ros_comm/test/test_ros/nodes/listener.py
--- ros_comm-1.6.6/test/test_ros/nodes/listener.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/nodes/listener.py	2012-04-15 22:54:41.777888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/nodes/talker.py ros_comm/test/test_ros/nodes/talker.py
--- ros_comm-1.6.6/test/test_ros/nodes/talker.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/nodes/talker.py	2012-04-15 22:54:41.784554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/nodes/testAllCommonFlows ros_comm/test/test_ros/nodes/testAllCommonFlows
--- ros_comm-1.6.6/test/test_ros/nodes/testAllCommonFlows	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/nodes/testAllCommonFlows	2012-04-15 22:54:41.797888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PKG = 'test_ros'
 MODULE = 'test_ros.testAllCommonFlows' #this is the module that will be invoked by python
diff -Naur ros_comm-1.6.6/test/test_ros/nodes/testMaster ros_comm/test/test_ros/nodes/testMaster
--- ros_comm-1.6.6/test/test_ros/nodes/testMaster	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/nodes/testMaster	2012-04-15 22:54:41.787888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PKG = 'test_ros'
 MODULE = 'test_ros.testMaster' #this is the module that will be invoked by python
diff -Naur ros_comm-1.6.6/test/test_ros/nodes/testSlave ros_comm/test/test_ros/nodes/testSlave
--- ros_comm-1.6.6/test/test_ros/nodes/testSlave	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/nodes/testSlave	2012-04-15 22:54:41.774554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PKG = 'test_ros'
 MODULE = 'test_ros.testSlave' #this is the module that will be invoked by python
diff -Naur ros_comm-1.6.6/test/test_ros/src/test_ros/param_server.py ros_comm/test/test_ros/src/test_ros/param_server.py
--- ros_comm-1.6.6/test/test_ros/src/test_ros/param_server.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/src/test_ros/param_server.py	2012-04-15 22:54:41.804554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/src/test_ros/rosclient.py ros_comm/test/test_ros/src/test_ros/rosclient.py
--- ros_comm-1.6.6/test/test_ros/src/test_ros/rosclient.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/src/test_ros/rosclient.py	2012-04-15 22:54:41.807888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/test/client_verification/test_slave_api.py ros_comm/test/test_ros/test/client_verification/test_slave_api.py
--- ros_comm-1.6.6/test/test_ros/test/client_verification/test_slave_api.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/test/client_verification/test_slave_api.py	2012-04-15 22:54:41.631221466 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/test/test_master_api.py ros_comm/test/test_ros/test/test_master_api.py
--- ros_comm-1.6.6/test/test_ros/test/test_master_api.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/test/test_master_api.py	2012-04-15 22:54:41.767888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/test/test_node_api.py ros_comm/test/test_ros/test/test_node_api.py
--- ros_comm-1.6.6/test/test_ros/test/test_node_api.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/test/test_node_api.py	2012-04-15 22:54:41.654554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/test/test_ps_encapsulation.py ros_comm/test/test_ros/test/test_ps_encapsulation.py
--- ros_comm-1.6.6/test/test_ros/test/test_ps_encapsulation.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/test/test_ps_encapsulation.py	2012-04-15 22:54:41.637888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/test/test_ps_get_param.py ros_comm/test/test_ros/test/test_ps_get_param.py
--- ros_comm-1.6.6/test/test_ros/test/test_ps_get_param.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/test/test_ps_get_param.py	2012-04-15 22:54:41.644554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/test/test_ps_has_param.py ros_comm/test/test_ros/test/test_ps_has_param.py
--- ros_comm-1.6.6/test/test_ros/test/test_ps_has_param.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/test/test_ps_has_param.py	2012-04-15 22:54:41.634554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/test/test_ps_private_names.py ros_comm/test/test_ros/test/test_ps_private_names.py
--- ros_comm-1.6.6/test/test_ros/test/test_ps_private_names.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/test/test_ps_private_names.py	2012-04-15 22:54:41.757888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/test/test_ps_scope_down.py ros_comm/test/test_ros/test/test_ps_scope_down.py
--- ros_comm-1.6.6/test/test_ros/test/test_ps_scope_down.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/test/test_ps_scope_down.py	2012-04-15 22:54:41.641221466 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/test/test_ps_scope_up.py ros_comm/test/test_ros/test/test_ps_scope_up.py
--- ros_comm-1.6.6/test/test_ros/test/test_ps_scope_up.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/test/test_ps_scope_up.py	2012-04-15 22:54:41.761221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/test/test_ps_search_param.py ros_comm/test/test_ros/test/test_ps_search_param.py
--- ros_comm-1.6.6/test/test_ros/test/test_ps_search_param.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/test/test_ps_search_param.py	2012-04-15 22:54:41.771221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/test/test_ps_set_param.py ros_comm/test/test_ros/test/test_ps_set_param.py
--- ros_comm-1.6.6/test/test_ros/test/test_ps_set_param.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/test/test_ps_set_param.py	2012-04-15 22:54:41.764554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_ros/test/test_ps_values.py ros_comm/test/test_ros/test/test_ps_values.py
--- ros_comm-1.6.6/test/test_ros/test/test_ps_values.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_ros/test/test_ps_values.py	2012-04-15 22:54:41.627888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosbag/scripts/generate_data_1.py ros_comm/test/test_rosbag/scripts/generate_data_1.py
--- ros_comm-1.6.6/test/test_rosbag/scripts/generate_data_1.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rosbag/scripts/generate_data_1.py	2012-04-15 22:54:41.837888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosbag/scripts/generate_data_2.py ros_comm/test/test_rosbag/scripts/generate_data_2.py
--- ros_comm-1.6.6/test/test_rosbag/scripts/generate_data_2.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rosbag/scripts/generate_data_2.py	2012-04-15 22:54:41.834554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosbag/scripts/generate_data_3.py ros_comm/test/test_rosbag/scripts/generate_data_3.py
--- ros_comm-1.6.6/test/test_rosbag/scripts/generate_data_3.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rosbag/scripts/generate_data_3.py	2012-04-15 22:54:41.831221464 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosbag/test/migrate_test.py ros_comm/test/test_rosbag/test/migrate_test.py
--- ros_comm-1.6.6/test/test_rosbag/test/migrate_test.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rosbag/test/migrate_test.py	2012-04-15 22:54:41.847888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosbag/test/random_play.py ros_comm/test/test_rosbag/test/random_play.py
--- ros_comm-1.6.6/test/test_rosbag/test/random_play.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rosbag/test/random_play.py	2012-04-15 22:54:41.851221464 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosbag/test/random_record.py ros_comm/test/test_rosbag/test/random_record.py
--- ros_comm-1.6.6/test/test_rosbag/test/random_record.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rosbag/test/random_record.py	2012-04-15 22:54:41.841221464 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosgraph/test/test_rosgraph_command_offline.py ros_comm/test/test_rosgraph/test/test_rosgraph_command_offline.py
--- ros_comm-1.6.6/test/test_rosgraph/test/test_rosgraph_command_offline.py	2011-12-08 17:13:53.000000000 +0100
+++ ros_comm/test/test_rosgraph/test/test_rosgraph_command_offline.py	2012-04-15 22:54:42.021221464 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosgraph/test/test_rosgraph_masterapi_online.py ros_comm/test/test_rosgraph/test/test_rosgraph_masterapi_online.py
--- ros_comm-1.6.6/test/test_rosgraph/test/test_rosgraph_masterapi_online.py	2011-12-08 17:13:53.000000000 +0100
+++ ros_comm/test/test_rosgraph/test/test_rosgraph_masterapi_online.py	2012-04-15 22:54:42.024554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/env.py ros_comm/test/test_roslaunch/test/env.py
--- ros_comm-1.6.6/test/test_roslaunch/test/env.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/env.py	2012-04-15 22:54:42.071221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/params_basic.py ros_comm/test/test_roslaunch/test/params_basic.py
--- ros_comm-1.6.6/test/test_roslaunch/test/params_basic.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/params_basic.py	2012-04-15 22:54:42.041221463 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_core.py ros_comm/test/test_roslaunch/test/test_core.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_core.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_core.py	2012-04-15 22:54:42.081221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_nodeprocess.py ros_comm/test/test_roslaunch/test/test_nodeprocess.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_nodeprocess.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_nodeprocess.py	2012-04-15 22:54:42.061221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_child.py ros_comm/test/test_roslaunch/test/test_roslaunch_child.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_child.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_roslaunch_child.py	2012-04-15 22:54:42.067888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_command_line_online.py ros_comm/test/test_roslaunch/test/test_roslaunch_command_line_online.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_command_line_online.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_roslaunch_command_line_online.py	2012-04-15 22:54:42.044554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_dump_params.py ros_comm/test/test_roslaunch/test/test_roslaunch_dump_params.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_dump_params.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_roslaunch_dump_params.py	2012-04-15 22:54:42.037888130 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2010, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_launch.py ros_comm/test/test_roslaunch/test/test_roslaunch_launch.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_launch.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_roslaunch_launch.py	2012-04-15 22:54:42.057888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_list_files.py ros_comm/test/test_roslaunch/test/test_roslaunch_list_files.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_list_files.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_roslaunch_list_files.py	2012-04-15 22:54:42.077888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2011, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_node_args.py ros_comm/test/test_roslaunch/test/test_roslaunch_node_args.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_node_args.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_roslaunch_node_args.py	2012-04-15 22:54:42.074554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_parent.py ros_comm/test/test_roslaunch/test/test_roslaunch_parent.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_parent.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_roslaunch_parent.py	2012-04-15 22:54:42.064554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_pmon.py ros_comm/test/test_roslaunch/test/test_roslaunch_pmon.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_pmon.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_roslaunch_pmon.py	2012-04-15 22:54:42.091221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_remote.py ros_comm/test/test_roslaunch/test/test_roslaunch_remote.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_remote.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_roslaunch_remote.py	2012-04-15 22:54:42.054554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_rlutil.py ros_comm/test/test_roslaunch/test/test_roslaunch_rlutil.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_rlutil.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_roslaunch_rlutil.py	2012-04-15 22:54:42.097888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_server.py ros_comm/test/test_roslaunch/test/test_roslaunch_server.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_roslaunch_server.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_roslaunch_server.py	2012-04-15 22:54:42.051221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_version.py ros_comm/test/test_roslaunch/test/test_version.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_version.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_version.py	2012-04-15 22:54:42.094554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2011, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslaunch/test/test_xmlloader.py ros_comm/test/test_roslaunch/test/test_xmlloader.py
--- ros_comm-1.6.6/test/test_roslaunch/test/test_xmlloader.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslaunch/test/test_xmlloader.py	2012-04-15 22:54:42.087888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslib_comm/test/test_md5sums.py ros_comm/test/test_roslib_comm/test/test_md5sums.py
--- ros_comm-1.6.6/test/test_roslib_comm/test/test_md5sums.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslib_comm/test/test_md5sums.py	2012-04-15 22:54:42.027888130 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslib_comm/test/test_roslib_genpy.py ros_comm/test/test_roslib_comm/test/test_roslib_genpy.py
--- ros_comm-1.6.6/test/test_roslib_comm/test/test_roslib_genpy.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslib_comm/test/test_roslib_genpy.py	2012-04-15 22:54:42.031221464 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roslib_comm/test/test_roslib_gentools.py ros_comm/test/test_roslib_comm/test/test_roslib_gentools.py
--- ros_comm-1.6.6/test/test_roslib_comm/test/test_roslib_gentools.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_roslib_comm/test/test_roslib_gentools.py	2012-04-15 22:54:42.034554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosmaster/test/test_rosmaster_paramserver.py ros_comm/test/test_rosmaster/test/test_rosmaster_paramserver.py
--- ros_comm-1.6.6/test/test_rosmaster/test/test_rosmaster_paramserver.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rosmaster/test/test_rosmaster_paramserver.py	2012-04-15 22:54:42.117888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosmaster/test/test_rosmaster_registrations.py ros_comm/test/test_rosmaster/test/test_rosmaster_registrations.py
--- ros_comm-1.6.6/test/test_rosmaster/test/test_rosmaster_registrations.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rosmaster/test/test_rosmaster_registrations.py	2012-04-15 22:54:42.114554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosmaster/test/test_rosmaster_validators.py ros_comm/test/test_rosmaster/test/test_rosmaster_validators.py
--- ros_comm-1.6.6/test/test_rosmaster/test/test_rosmaster_validators.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rosmaster/test/test_rosmaster_validators.py	2012-04-15 22:54:42.107888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosmsg/test/test_rosmsg_command_line.py ros_comm/test/test_rosmsg/test/test_rosmsg_command_line.py
--- ros_comm-1.6.6/test/test_rosmsg/test/test_rosmsg_command_line.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rosmsg/test/test_rosmsg_command_line.py	2012-04-15 22:54:41.854554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosmsg/test/test_rosmsg.py ros_comm/test/test_rosmsg/test/test_rosmsg.py
--- ros_comm-1.6.6/test/test_rosmsg/test/test_rosmsg.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rosmsg/test/test_rosmsg.py	2012-04-15 22:54:41.857888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosnode/test/test_rosnode_command_offline.py ros_comm/test/test_rosnode/test/test_rosnode_command_offline.py
--- ros_comm-1.6.6/test/test_rosnode/test/test_rosnode_command_offline.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rosnode/test/test_rosnode_command_offline.py	2012-04-15 22:54:42.127888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosnode/test/test_rosnode_command_online.py ros_comm/test/test_rosnode/test/test_rosnode_command_online.py
--- ros_comm-1.6.6/test/test_rosnode/test/test_rosnode_command_online.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rosnode/test/test_rosnode_command_online.py	2012-04-15 22:54:42.124554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosnode/test/test_rosnode.py ros_comm/test/test_rosnode/test/test_rosnode.py
--- ros_comm-1.6.6/test/test_rosnode/test/test_rosnode.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rosnode/test/test_rosnode.py	2012-04-15 22:54:42.121221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosparam/test/test_rosparam_command_line_offline.py ros_comm/test/test_rosparam/test/test_rosparam_command_line_offline.py
--- ros_comm-1.6.6/test/test_rosparam/test/test_rosparam_command_line_offline.py	2011-12-08 17:13:53.000000000 +0100
+++ ros_comm/test/test_rosparam/test/test_rosparam_command_line_offline.py	2012-04-15 22:54:41.811221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosparam/test/test_rosparam_command_line_online.py ros_comm/test/test_rosparam/test/test_rosparam_command_line_online.py
--- ros_comm-1.6.6/test/test_rosparam/test/test_rosparam_command_line_online.py	2011-12-08 17:13:53.000000000 +0100
+++ ros_comm/test/test_rosparam/test/test_rosparam_command_line_online.py	2012-04-15 22:54:41.817888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosparam/test/test_rosparam.py ros_comm/test/test_rosparam/test/test_rosparam.py
--- ros_comm-1.6.6/test/test_rosparam/test/test_rosparam.py	2011-12-08 17:13:53.000000000 +0100
+++ ros_comm/test/test_rosparam/test/test_rosparam.py	2012-04-15 22:54:41.814554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/nodes/listener.py ros_comm/test/test_rospy/nodes/listener.py
--- ros_comm-1.6.6/test/test_rospy/nodes/listener.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/nodes/listener.py	2012-04-15 22:54:41.997888130 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/nodes/publish_on_shutdown.py ros_comm/test/test_rospy/nodes/publish_on_shutdown.py
--- ros_comm-1.6.6/test/test_rospy/nodes/publish_on_shutdown.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/nodes/publish_on_shutdown.py	2012-04-15 22:54:41.994554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/nodes/talker.py ros_comm/test/test_rospy/nodes/talker.py
--- ros_comm-1.6.6/test/test_rospy/nodes/talker.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/nodes/talker.py	2012-04-15 22:54:42.001221464 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/listenerpublisher_embed.py ros_comm/test/test_rospy/test/listenerpublisher_embed.py
--- ros_comm-1.6.6/test/test_rospy/test/listenerpublisher_embed.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/listenerpublisher_embed.py	2012-04-15 22:54:41.934554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/listenerpublisher.py ros_comm/test/test_rospy/test/listenerpublisher.py
--- ros_comm-1.6.6/test/test_rospy/test/listenerpublisher.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/listenerpublisher.py	2012-04-15 22:54:41.911221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/talker ros_comm/test/test_rospy/test/talker
--- ros_comm-1.6.6/test/test_rospy/test/talker	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/talker	2012-04-15 22:54:41.991221464 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_basic_services.py ros_comm/test/test_rospy/test/test_basic_services.py
--- ros_comm-1.6.6/test/test_rospy/test/test_basic_services.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_basic_services.py	2012-04-15 22:54:41.987888130 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_client_param_api.py ros_comm/test/test_rospy/test/test_client_param_api.py
--- ros_comm-1.6.6/test/test_rospy/test/test_client_param_api.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_client_param_api.py	2012-04-15 22:54:41.914554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_client_param_server.py ros_comm/test/test_rospy/test/test_client_param_server.py
--- ros_comm-1.6.6/test/test_rospy/test/test_client_param_server.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_client_param_server.py	2012-04-15 22:54:41.984554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_deregister.py ros_comm/test/test_rospy/test/test_deregister.py
--- ros_comm-1.6.6/test/test_rospy/test/test_deregister.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_deregister.py	2012-04-15 22:54:41.981221464 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_embed_msg.py ros_comm/test/test_rospy/test/test_embed_msg.py
--- ros_comm-1.6.6/test/test_rospy/test/test_embed_msg.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_embed_msg.py	2012-04-15 22:54:41.974554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_empty_service.py ros_comm/test/test_rospy/test/test_empty_service.py
--- ros_comm-1.6.6/test/test_rospy/test/test_empty_service.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_empty_service.py	2012-04-15 22:54:41.904554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_genmsg_py.py ros_comm/test/test_rospy/test/test_genmsg_py.py
--- ros_comm-1.6.6/test/test_rospy/test/test_genmsg_py.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_genmsg_py.py	2012-04-15 22:54:41.871221464 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_gensrv_py.py ros_comm/test/test_rospy/test/test_gensrv_py.py
--- ros_comm-1.6.6/test/test_rospy/test/test_gensrv_py.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_gensrv_py.py	2012-04-15 22:54:41.931221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_latch.py ros_comm/test/test_rospy/test/test_latch.py
--- ros_comm-1.6.6/test/test_rospy/test/test_latch.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_latch.py	2012-04-15 22:54:41.924554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_node.py ros_comm/test/test_rospy/test/test_node.py
--- ros_comm-1.6.6/test/test_rospy/test/test_node.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_node.py	2012-04-15 22:54:41.944554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_on_shutdown.py ros_comm/test/test_rospy/test/test_on_shutdown.py
--- ros_comm-1.6.6/test/test_rospy/test/test_on_shutdown.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_on_shutdown.py	2012-04-15 22:54:41.874554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_pubsub_order.py ros_comm/test/test_rospy/test/test_pubsub_order.py
--- ros_comm-1.6.6/test/test_rospy/test/test_pubsub_order.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_pubsub_order.py	2012-04-15 22:54:41.927888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_api.py ros_comm/test/test_rospy/test/test_rospy_api.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_api.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_api.py	2012-04-15 22:54:41.894554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_client_online.py ros_comm/test/test_rospy/test/test_rospy_client_online.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_client_online.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_client_online.py	2012-04-15 22:54:41.977888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_client.py ros_comm/test/test_rospy/test/test_rospy_client.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_client.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_client.py	2012-04-15 22:54:41.967888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_core.py ros_comm/test/test_rospy/test/test_rospy_core.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_core.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_core.py	2012-04-15 22:54:41.891221464 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_exceptions.py ros_comm/test/test_rospy/test/test_rospy_exceptions.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_exceptions.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_exceptions.py	2012-04-15 22:54:41.864554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_msg.py ros_comm/test/test_rospy/test/test_rospy_msg.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_msg.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_msg.py	2012-04-15 22:54:41.964554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_msnode.py ros_comm/test/test_rospy/test/test_rospy_msnode.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_msnode.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_msnode.py	2012-04-15 22:54:41.907888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_names.py ros_comm/test/test_rospy/test/test_rospy_names.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_names.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_names.py	2012-04-15 22:54:41.921221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_numpy.py ros_comm/test/test_rospy/test/test_rospy_numpy.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_numpy.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_numpy.py	2012-04-15 22:54:41.951221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_paramserver.py ros_comm/test/test_rospy/test/test_rospy_paramserver.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_paramserver.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_paramserver.py	2012-04-15 22:54:41.884554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_registration.py ros_comm/test/test_rospy/test/test_rospy_registration.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_registration.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_registration.py	2012-04-15 22:54:41.901221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_rostime.py ros_comm/test/test_rospy/test/test_rospy_rostime.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_rostime.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_rostime.py	2012-04-15 22:54:41.897888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_service.py ros_comm/test/test_rospy/test/test_rospy_service.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_service.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_service.py	2012-04-15 22:54:41.961221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_tcpros_base.py ros_comm/test/test_rospy/test/test_rospy_tcpros_base.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_tcpros_base.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_tcpros_base.py	2012-04-15 22:54:41.957888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_tcpros_pubsub.py ros_comm/test/test_rospy/test/test_rospy_tcpros_pubsub.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_tcpros_pubsub.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_tcpros_pubsub.py	2012-04-15 22:54:41.947888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_tcpros.py ros_comm/test/test_rospy/test/test_rospy_tcpros.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_tcpros.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_tcpros.py	2012-04-15 22:54:41.917888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_tcpros_service.py ros_comm/test/test_rospy/test/test_rospy_tcpros_service.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_tcpros_service.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_tcpros_service.py	2012-04-15 22:54:41.937888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_timer_online.py ros_comm/test/test_rospy/test/test_rospy_timer_online.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_timer_online.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_timer_online.py	2012-04-15 22:54:41.941221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_topics.py ros_comm/test/test_rospy/test/test_rospy_topics.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_topics.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_topics.py	2012-04-15 22:54:41.861221464 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_transport.py ros_comm/test/test_rospy/test/test_rospy_transport.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_transport.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_transport.py	2012-04-15 22:54:41.877888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_rospy_validators.py ros_comm/test/test_rospy/test/test_rospy_validators.py
--- ros_comm-1.6.6/test/test_rospy/test/test_rospy_validators.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_rospy_validators.py	2012-04-15 22:54:41.867888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_service_failure.py ros_comm/test/test_rospy/test/test_service_failure.py
--- ros_comm-1.6.6/test/test_rospy/test/test_service_failure.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_service_failure.py	2012-04-15 22:54:41.881221464 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rospy/test/test_service_order.py ros_comm/test/test_rospy/test/test_service_order.py
--- ros_comm-1.6.6/test/test_rospy/test/test_service_order.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rospy/test/test_service_order.py	2012-04-15 22:54:41.971221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosservice/nodes/header_echo_server.py ros_comm/test/test_rosservice/nodes/header_echo_server.py
--- ros_comm-1.6.6/test/test_rosservice/nodes/header_echo_server.py	2011-12-08 17:13:53.000000000 +0100
+++ ros_comm/test/test_rosservice/nodes/header_echo_server.py	2012-04-15 22:54:41.617888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosservice/test/test_rosservice_command_line_offline.py ros_comm/test/test_rosservice/test/test_rosservice_command_line_offline.py
--- ros_comm-1.6.6/test/test_rosservice/test/test_rosservice_command_line_offline.py	2011-12-08 17:13:53.000000000 +0100
+++ ros_comm/test/test_rosservice/test/test_rosservice_command_line_offline.py	2012-04-15 22:54:41.604554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosservice/test/test_rosservice_command_line_online.py ros_comm/test/test_rosservice/test/test_rosservice_command_line_online.py
--- ros_comm-1.6.6/test/test_rosservice/test/test_rosservice_command_line_online.py	2011-12-08 17:13:53.000000000 +0100
+++ ros_comm/test/test_rosservice/test/test_rosservice_command_line_online.py	2012-04-15 22:54:41.614554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rosservice/test/test_rosservice.py ros_comm/test/test_rosservice/test/test_rosservice.py
--- ros_comm-1.6.6/test/test_rosservice/test/test_rosservice.py	2011-12-08 17:13:53.000000000 +0100
+++ ros_comm/test/test_rosservice/test/test_rosservice.py	2012-04-15 22:54:41.611221466 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rostest/test/test_clean_master.py ros_comm/test/test_rostest/test/test_clean_master.py
--- ros_comm-1.6.6/test/test_rostest/test/test_clean_master.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rostest/test/test_clean_master.py	2012-04-15 22:54:42.104554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rostest/test/time_limit_test.py ros_comm/test/test_rostest/test/time_limit_test.py
--- ros_comm-1.6.6/test/test_rostest/test/time_limit_test.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rostest/test/time_limit_test.py	2012-04-15 22:54:42.101221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rostopic/test/test_rostopic_command_line_offline.py ros_comm/test/test_rostopic/test/test_rostopic_command_line_offline.py
--- ros_comm-1.6.6/test/test_rostopic/test/test_rostopic_command_line_offline.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rostopic/test/test_rostopic_command_line_offline.py	2012-04-15 22:54:42.014554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rostopic/test/test_rostopic_command_line_online.py ros_comm/test/test_rostopic/test/test_rostopic_command_line_online.py
--- ros_comm-1.6.6/test/test_rostopic/test/test_rostopic_command_line_online.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rostopic/test/test_rostopic_command_line_online.py	2012-04-15 22:54:42.011221464 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rostopic/test/test_rostopic.py ros_comm/test/test_rostopic/test/test_rostopic.py
--- ros_comm-1.6.6/test/test_rostopic/test/test_rostopic.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rostopic/test/test_rostopic.py	2012-04-15 22:54:42.004554797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_rostopic/test/test_rostopic_unit.py ros_comm/test/test_rostopic/test/test_rostopic_unit.py
--- ros_comm-1.6.6/test/test_rostopic/test/test_rostopic_unit.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/test/test_rostopic/test/test_rostopic_unit.py	2012-04-15 22:54:42.017888130 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roswtf/test/test_roswtf_command_line_offline.py ros_comm/test/test_roswtf/test/test_roswtf_command_line_offline.py
--- ros_comm-1.6.6/test/test_roswtf/test/test_roswtf_command_line_offline.py	2011-12-08 17:13:53.000000000 +0100
+++ ros_comm/test/test_roswtf/test/test_roswtf_command_line_offline.py	2012-04-15 22:54:41.621221466 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_roswtf/test/test_roswtf_command_line_online.py ros_comm/test/test_roswtf/test/test_roswtf_command_line_online.py
--- ros_comm-1.6.6/test/test_roswtf/test/test_roswtf_command_line_online.py	2011-12-08 17:13:53.000000000 +0100
+++ ros_comm/test/test_roswtf/test/test_roswtf_command_line_online.py	2012-04-15 22:54:41.624554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/test/test_topic_tools/test/test_mux_delete_add.py ros_comm/test/test_topic_tools/test/test_mux_delete_add.py
--- ros_comm-1.6.6/test/test_topic_tools/test/test_mux_delete_add.py	2011-12-08 17:13:53.000000000 +0100
+++ ros_comm/test/test_topic_tools/test/test_mux_delete_add.py	2012-04-15 22:54:41.824554798 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 # Software License Agreement (BSD License)
 #
diff -Naur ros_comm-1.6.6/test/test_topic_tools/test/test_mux_services.py ros_comm/test/test_topic_tools/test/test_mux_services.py
--- ros_comm-1.6.6/test/test_topic_tools/test/test_mux_services.py	2011-12-08 17:13:53.000000000 +0100
+++ ros_comm/test/test_topic_tools/test/test_mux_services.py	2012-04-15 22:54:41.827888131 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 # Software License Agreement (BSD License)
 #
diff -Naur ros_comm-1.6.6/test/test_topic_tools/test/test_one_message.py ros_comm/test/test_topic_tools/test/test_one_message.py
--- ros_comm-1.6.6/test/test_topic_tools/test/test_one_message.py	2011-12-08 17:13:53.000000000 +0100
+++ ros_comm/test/test_topic_tools/test/test_one_message.py	2012-04-15 22:54:41.821221465 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/bin/rosbag ros_comm/tools/rosbag/bin/rosbag
--- ros_comm-1.6.6/tools/rosbag/bin/rosbag	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/bin/rosbag	2012-04-15 22:54:41.517888134 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/scripts/bag2png.py ros_comm/tools/rosbag/scripts/bag2png.py
--- ros_comm-1.6.6/tools/rosbag/scripts/bag2png.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/scripts/bag2png.py	2012-04-15 22:54:41.501221462 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/scripts/bagsort.py ros_comm/tools/rosbag/scripts/bagsort.py
--- ros_comm-1.6.6/tools/rosbag/scripts/bagsort.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/scripts/bagsort.py	2012-04-15 22:54:41.507888128 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/scripts/fastrebag.py ros_comm/tools/rosbag/scripts/fastrebag.py
--- ros_comm-1.6.6/tools/rosbag/scripts/fastrebag.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/scripts/fastrebag.py	2012-04-15 22:54:41.514554800 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/scripts/fixbag_batch.py ros_comm/tools/rosbag/scripts/fixbag_batch.py
--- ros_comm-1.6.6/tools/rosbag/scripts/fixbag_batch.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/scripts/fixbag_batch.py	2012-04-15 22:54:41.504554793 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/scripts/fixbag.py ros_comm/tools/rosbag/scripts/fixbag.py
--- ros_comm-1.6.6/tools/rosbag/scripts/fixbag.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/scripts/fixbag.py	2012-04-15 22:54:41.494554810 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/scripts/fix_md5sums.py ros_comm/tools/rosbag/scripts/fix_md5sums.py
--- ros_comm-1.6.6/tools/rosbag/scripts/fix_md5sums.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/scripts/fix_md5sums.py	2012-04-15 22:54:41.497888137 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/scripts/fix_moved_messages.py ros_comm/tools/rosbag/scripts/fix_moved_messages.py
--- ros_comm-1.6.6/tools/rosbag/scripts/fix_moved_messages.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/scripts/fix_moved_messages.py	2012-04-15 22:54:41.481221423 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/scripts/fix_msg_defs.py ros_comm/tools/rosbag/scripts/fix_msg_defs.py
--- ros_comm-1.6.6/tools/rosbag/scripts/fix_msg_defs.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/scripts/fix_msg_defs.py	2012-04-15 22:54:41.491221484 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/scripts/makerule.py ros_comm/tools/rosbag/scripts/makerule.py
--- ros_comm-1.6.6/tools/rosbag/scripts/makerule.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/scripts/makerule.py	2012-04-15 22:54:41.511221466 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/scripts/savemsg.py ros_comm/tools/rosbag/scripts/savemsg.py
--- ros_comm-1.6.6/tools/rosbag/scripts/savemsg.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/scripts/savemsg.py	2012-04-15 22:54:41.487888150 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/scripts/topic_renamer.py ros_comm/tools/rosbag/scripts/topic_renamer.py
--- ros_comm-1.6.6/tools/rosbag/scripts/topic_renamer.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/scripts/topic_renamer.py	2012-04-15 22:54:41.484554788 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/test/connection_count.py ros_comm/tools/rosbag/test/connection_count.py
--- ros_comm-1.6.6/tools/rosbag/test/connection_count.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/test/connection_count.py	2012-04-15 22:54:41.524554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/test/latched_pub.py ros_comm/tools/rosbag/test/latched_pub.py
--- ros_comm-1.6.6/tools/rosbag/test/latched_pub.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/test/latched_pub.py	2012-04-15 22:54:41.534554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/test/latched_sub.py ros_comm/tools/rosbag/test/latched_sub.py
--- ros_comm-1.6.6/tools/rosbag/test/latched_sub.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/test/latched_sub.py	2012-04-15 22:54:41.527888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosbag/test/test_bag.py ros_comm/tools/rosbag/test/test_bag.py
--- ros_comm-1.6.6/tools/rosbag/test/test_bag.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/test/test_bag.py	2012-04-15 22:54:41.531221466 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 #
 # test_bag.py
 
diff -Naur ros_comm-1.6.6/tools/rosbag/test/topic_count.py ros_comm/tools/rosbag/test/topic_count.py
--- ros_comm-1.6.6/tools/rosbag/test/topic_count.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosbag/test/topic_count.py	2012-04-15 22:54:41.521221467 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosgraph/nodes/rosgraph ros_comm/tools/rosgraph/nodes/rosgraph
--- ros_comm-1.6.6/tools/rosgraph/nodes/rosgraph	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosgraph/nodes/rosgraph	2012-04-15 22:54:41.564554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosgraph/nodes/rxgraph ros_comm/tools/rosgraph/nodes/rxgraph
--- ros_comm-1.6.6/tools/rosgraph/nodes/rxgraph	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosgraph/nodes/rxgraph	2012-04-15 22:54:41.561221466 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosgraph/src/rosgraph/rosgraph_main.py ros_comm/tools/rosgraph/src/rosgraph/rosgraph_main.py
--- ros_comm-1.6.6/tools/rosgraph/src/rosgraph/rosgraph_main.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosgraph/src/rosgraph/rosgraph_main.py	2012-04-15 22:54:41.567888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/roslaunch/bin/roslaunch ros_comm/tools/roslaunch/bin/roslaunch
--- ros_comm-1.6.6/tools/roslaunch/bin/roslaunch	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/roslaunch/bin/roslaunch	2012-04-15 22:54:41.601221466 +0200
@@ -1,3 +1,3 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 from ros import roslaunch
 roslaunch.main()
diff -Naur ros_comm-1.6.6/tools/roslaunch/bin/roslaunch-console ros_comm/tools/roslaunch/bin/roslaunch-console
--- ros_comm-1.6.6/tools/roslaunch/bin/roslaunch-console	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/roslaunch/bin/roslaunch-console	2012-04-15 22:54:41.597888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import roslib; roslib.load_manifest('roslaunch')
 
diff -Naur ros_comm-1.6.6/tools/rosnode/src/rosnode.py ros_comm/tools/rosnode/src/rosnode.py
--- ros_comm-1.6.6/tools/rosnode/src/rosnode.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosnode/src/rosnode.py	2012-04-15 22:54:41.544554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosparam/src/rosparam.py ros_comm/tools/rosparam/src/rosparam.py
--- ros_comm-1.6.6/tools/rosparam/src/rosparam.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosparam/src/rosparam.py	2012-04-15 22:54:41.537888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosservice/bin/rosservice ros_comm/tools/rosservice/bin/rosservice
--- ros_comm-1.6.6/tools/rosservice/bin/rosservice	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosservice/bin/rosservice	2012-04-15 22:54:41.571221466 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rosservice/src/rosservice.py ros_comm/tools/rosservice/src/rosservice.py
--- ros_comm-1.6.6/tools/rosservice/src/rosservice.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rosservice/src/rosservice.py	2012-04-15 22:54:41.574554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rostest/bin/roslaunch-check.py ros_comm/tools/rostest/bin/roslaunch-check.py
--- ros_comm-1.6.6/tools/rostest/bin/roslaunch-check.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rostest/bin/roslaunch-check.py	2012-04-15 22:54:41.587888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rostest/bin/rostest ros_comm/tools/rostest/bin/rostest
--- ros_comm-1.6.6/tools/rostest/bin/rostest	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rostest/bin/rostest	2012-04-15 22:54:41.591221466 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rostest/nodes/hztest ros_comm/tools/rostest/nodes/hztest
--- ros_comm-1.6.6/tools/rostest/nodes/hztest	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rostest/nodes/hztest	2012-04-15 22:54:41.594554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/rostopic/bin/rostopic ros_comm/tools/rostopic/bin/rostopic
--- ros_comm-1.6.6/tools/rostopic/bin/rostopic	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rostopic/bin/rostopic	2012-04-15 22:54:41.577888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import roslib; roslib.load_manifest('rostopic')
 import rostopic
 rostopic.rostopicmain()
diff -Naur ros_comm-1.6.6/tools/rostopic/src/rostopic.py ros_comm/tools/rostopic/src/rostopic.py
--- ros_comm-1.6.6/tools/rostopic/src/rostopic.py	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/rostopic/src/rostopic.py	2012-04-15 22:54:41.584554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/topic_tools/scripts/mux_add ros_comm/tools/topic_tools/scripts/mux_add
--- ros_comm-1.6.6/tools/topic_tools/scripts/mux_add	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/topic_tools/scripts/mux_add	2012-04-15 22:54:41.547888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/topic_tools/scripts/mux_delete ros_comm/tools/topic_tools/scripts/mux_delete
--- ros_comm-1.6.6/tools/topic_tools/scripts/mux_delete	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/topic_tools/scripts/mux_delete	2012-04-15 22:54:41.557888132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/topic_tools/scripts/mux_list ros_comm/tools/topic_tools/scripts/mux_list
--- ros_comm-1.6.6/tools/topic_tools/scripts/mux_list	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/topic_tools/scripts/mux_list	2012-04-15 22:54:41.554554799 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/tools/topic_tools/scripts/mux_select ros_comm/tools/topic_tools/scripts/mux_select
--- ros_comm-1.6.6/tools/topic_tools/scripts/mux_select	2011-12-08 17:13:54.000000000 +0100
+++ ros_comm/tools/topic_tools/scripts/mux_select	2012-04-15 22:54:41.551221466 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur ros_comm-1.6.6/utilities/roswtf/bin/roswtf ros_comm/utilities/roswtf/bin/roswtf
--- ros_comm-1.6.6/utilities/roswtf/bin/roswtf	2011-12-08 17:13:55.000000000 +0100
+++ ros_comm/utilities/roswtf/bin/roswtf	2012-04-15 22:54:42.131221464 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 import roslib; roslib.load_manifest('roswtf')
 import roswtf
 
