diff -Naur pr2_common_actions-0.3.4/joint_trajectory_action_tools/src/joint_trajectory_action_tools/tools.py pr2_common_actions/joint_trajectory_action_tools/src/joint_trajectory_action_tools/tools.py
--- pr2_common_actions-0.3.4/joint_trajectory_action_tools/src/joint_trajectory_action_tools/tools.py	2011-03-21 20:03:38.000000000 +0100
+++ pr2_common_actions/joint_trajectory_action_tools/src/joint_trajectory_action_tools/tools.py	2012-05-08 14:56:04.853655286 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur pr2_common_actions-0.3.4/joint_trajectory_action_tools/test/test.py pr2_common_actions/joint_trajectory_action_tools/test/test.py
--- pr2_common_actions-0.3.4/joint_trajectory_action_tools/test/test.py	2011-03-21 20:03:38.000000000 +0100
+++ pr2_common_actions/joint_trajectory_action_tools/test/test.py	2012-05-08 14:56:04.850321953 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python  
+#!/usr/bin/env python2
 
 import roslib; roslib.load_manifest('pr2_plugs_actions')
 import rospy
diff -Naur pr2_common_actions-0.3.4/pr2_arm_move_ik/src/pr2_arm_move_ik/tools.py pr2_common_actions/pr2_arm_move_ik/src/pr2_arm_move_ik/tools.py
--- pr2_common_actions-0.3.4/pr2_arm_move_ik/src/pr2_arm_move_ik/tools.py	2011-03-21 20:03:38.000000000 +0100
+++ pr2_common_actions/pr2_arm_move_ik/src/pr2_arm_move_ik/tools.py	2012-05-08 14:56:04.866988615 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur pr2_common_actions-0.3.4/pr2_tuck_arms_action/scripts/tuck_arms.py pr2_common_actions/pr2_tuck_arms_action/scripts/tuck_arms.py
--- pr2_common_actions-0.3.4/pr2_tuck_arms_action/scripts/tuck_arms.py	2011-03-21 20:03:38.000000000 +0100
+++ pr2_common_actions/pr2_tuck_arms_action/scripts/tuck_arms.py	2012-05-08 14:56:04.856988617 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur pr2_common_actions-0.3.4/pr2_tuck_arms_action/src/pr2_tuck_arms_action/tuck_arms_main.py pr2_common_actions/pr2_tuck_arms_action/src/pr2_tuck_arms_action/tuck_arms_main.py
--- pr2_common_actions-0.3.4/pr2_tuck_arms_action/src/pr2_tuck_arms_action/tuck_arms_main.py	2011-03-21 20:03:38.000000000 +0100
+++ pr2_common_actions/pr2_tuck_arms_action/src/pr2_tuck_arms_action/tuck_arms_main.py	2012-05-08 14:56:04.863655281 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur pr2_common_actions-0.3.4/pr2_tuck_arms_action/test/tuck_arms_test.py pr2_common_actions/pr2_tuck_arms_action/test/tuck_arms_test.py
--- pr2_common_actions-0.3.4/pr2_tuck_arms_action/test/tuck_arms_test.py	2011-03-21 20:03:38.000000000 +0100
+++ pr2_common_actions/pr2_tuck_arms_action/test/tuck_arms_test.py	2012-05-08 14:56:04.860321948 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
