diff -Naur image_common-1.6.1-orig/image_transport/src/publisher.cpp image_common/image_transport/src/publisher.cpp
--- image_common-1.6.1-orig/image_transport/src/publisher.cpp	2011-12-06 02:43:44.000000000 +0100
+++ image_common/image_transport/src/publisher.cpp	2012-04-19 21:07:24.074375177 +0200
@@ -56,8 +56,10 @@
   uint32_t getNumSubscribers() const
   {
     uint32_t count = 0;
-    BOOST_FOREACH(const PublisherPlugin& pub, publishers_)
-      count += pub.getNumSubscribers();
+    for (boost::ptr_vector<PublisherPlugin>::const_iterator it = publishers_.begin();
+         it != publishers_.end(); it++) {
+      count += it->getNumSubscribers();
+    }
     return count;
   }
