diff -Naur diagnostics-1.6.4/diagnostic_aggregator/test/aggregator_test.py diagnostics/diagnostic_aggregator/test/aggregator_test.py
--- diagnostics-1.6.4/diagnostic_aggregator/test/aggregator_test.py	2012-03-16 22:23:21.000000000 +0100
+++ diagnostics/diagnostic_aggregator/test/aggregator_test.py	2012-04-17 19:43:06.201832768 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur diagnostics-1.6.4/diagnostic_aggregator/test/diag_pub.py diagnostics/diagnostic_aggregator/test/diag_pub.py
--- diagnostics-1.6.4/diagnostic_aggregator/test/diag_pub.py	2012-03-16 22:23:21.000000000 +0100
+++ diagnostics/diagnostic_aggregator/test/diag_pub.py	2012-04-17 19:43:06.198499434 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur diagnostics-1.6.4/diagnostic_aggregator/test/expected_stale_pub.py diagnostics/diagnostic_aggregator/test/expected_stale_pub.py
--- diagnostics-1.6.4/diagnostic_aggregator/test/expected_stale_pub.py	2012-03-16 22:23:21.000000000 +0100
+++ diagnostics/diagnostic_aggregator/test/expected_stale_pub.py	2012-04-17 19:43:06.195166100 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur diagnostics-1.6.4/diagnostic_aggregator/test/expected_stale_test.py diagnostics/diagnostic_aggregator/test/expected_stale_test.py
--- diagnostics-1.6.4/diagnostic_aggregator/test/expected_stale_test.py	2012-03-16 22:23:21.000000000 +0100
+++ diagnostics/diagnostic_aggregator/test/expected_stale_test.py	2012-04-17 19:43:06.205166102 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur diagnostics-1.6.4/diagnostic_aggregator/test/multiple_match_pub.py diagnostics/diagnostic_aggregator/test/multiple_match_pub.py
--- diagnostics-1.6.4/diagnostic_aggregator/test/multiple_match_pub.py	2012-03-16 22:23:21.000000000 +0100
+++ diagnostics/diagnostic_aggregator/test/multiple_match_pub.py	2012-04-17 19:43:06.208499435 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur diagnostics-1.6.4/diagnostic_aggregator/test/multiple_match_test.py diagnostics/diagnostic_aggregator/test/multiple_match_test.py
--- diagnostics-1.6.4/diagnostic_aggregator/test/multiple_match_test.py	2012-03-16 22:23:21.000000000 +0100
+++ diagnostics/diagnostic_aggregator/test/multiple_match_test.py	2012-04-17 19:43:06.191832767 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
diff -Naur diagnostics-1.6.4/self_test/scripts/test_selftest.py diagnostics/self_test/scripts/test_selftest.py
--- diagnostics-1.6.4/self_test/scripts/test_selftest.py	2012-03-16 22:23:21.000000000 +0100
+++ diagnostics/self_test/scripts/test_selftest.py	2012-04-17 19:43:06.188499434 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2010, Willow Garage, Inc.
diff -Naur diagnostics-1.6.4/test_diagnostic_aggregator/test/fail_init_test.py diagnostics/test_diagnostic_aggregator/test/fail_init_test.py
--- diagnostics-1.6.4/test_diagnostic_aggregator/test/fail_init_test.py	2012-03-16 22:23:21.000000000 +0100
+++ diagnostics/test_diagnostic_aggregator/test/fail_init_test.py	2012-04-17 19:43:06.211832768 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2010, Willow Garage, Inc.
diff -Naur diagnostics-1.6.4/test_diagnostic_aggregator/test/match_analyze_pub.py diagnostics/test_diagnostic_aggregator/test/match_analyze_pub.py
--- diagnostics-1.6.4/test_diagnostic_aggregator/test/match_analyze_pub.py	2012-03-16 22:23:21.000000000 +0100
+++ diagnostics/test_diagnostic_aggregator/test/match_analyze_pub.py	2012-04-17 19:43:06.218499435 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2010, Willow Garage, Inc.
diff -Naur diagnostics-1.6.4/test_diagnostic_aggregator/test/match_analyze_test.py diagnostics/test_diagnostic_aggregator/test/match_analyze_test.py
--- diagnostics-1.6.4/test_diagnostic_aggregator/test/match_analyze_test.py	2012-03-16 22:23:21.000000000 +0100
+++ diagnostics/test_diagnostic_aggregator/test/match_analyze_test.py	2012-04-17 19:43:06.215166102 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
