diff -Naur vision_opencv-1.6.13/opencv2/manifest.xml vision_opencv/opencv2/manifest.xml
--- vision_opencv-1.6.13/opencv2/manifest.xml	2012-04-05 12:14:09.000000000 +0200
+++ vision_opencv/opencv2/manifest.xml	2012-04-29 19:28:45.935114453 +0200
@@ -20,7 +20,7 @@
 <review status="3rdparty" notes=""/>
 <url>http://opencv.willowgarage.com</url>
 <export>
-  <cpp cflags="`pkg-config opencv-2.3.1 --cflags`" lflags="`pkg-config opencv-2.3.1 --libs`"/>
+  <cpp cflags="`pkg-config opencv --cflags`" lflags="`pkg-config opencv --libs`"/>
   <!-- On OS X there is a cmake Findopencv.cmake so pkg-config isn't necessary -->
   <cpp os="osx" cflags="`pkg-config opencv --cflags`" lflags="`pkg-config opencv --libs`"/>
 </export>
diff -Naur vision_opencv-1.6.13/opencv_tests/nodes/broadcast.py vision_opencv/opencv_tests/nodes/broadcast.py
--- vision_opencv-1.6.13/opencv_tests/nodes/broadcast.py	2012-04-05 12:14:09.000000000 +0200
+++ vision_opencv/opencv_tests/nodes/broadcast.py	2012-04-29 19:28:33.095114225 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur vision_opencv-1.6.13/opencv_tests/nodes/source.py vision_opencv/opencv_tests/nodes/source.py
--- vision_opencv-1.6.13/opencv_tests/nodes/source.py	2012-04-05 12:14:09.000000000 +0200
+++ vision_opencv/opencv_tests/nodes/source.py	2012-04-29 19:28:33.095114225 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
