diff -Naur pr2_arm_navigation-0.4.8/pr2_arm_navigation_actions/scripts/change_joint_controller_gains.py pr2_arm_navigation/pr2_arm_navigation_actions/scripts/change_joint_controller_gains.py
--- pr2_arm_navigation-0.4.8/pr2_arm_navigation_actions/scripts/change_joint_controller_gains.py	2012-01-06 20:23:20.000000000 +0100
+++ pr2_arm_navigation/pr2_arm_navigation_actions/scripts/change_joint_controller_gains.py	2012-05-08 14:35:27.363608626 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 import roslib; roslib.load_manifest('fingertip_reactive_grasp')
 import rospy
diff -Naur pr2_arm_navigation-0.4.8/pr2_arm_navigation_actions/scripts/clearAllObjects.py pr2_arm_navigation/pr2_arm_navigation_actions/scripts/clearAllObjects.py
--- pr2_arm_navigation-0.4.8/pr2_arm_navigation_actions/scripts/clearAllObjects.py	2012-01-06 20:23:20.000000000 +0100
+++ pr2_arm_navigation/pr2_arm_navigation_actions/scripts/clearAllObjects.py	2012-05-08 14:35:27.373608625 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'pr2_arm_navigation_actions'
 
diff -Naur pr2_arm_navigation-0.4.8/pr2_arm_navigation_actions/scripts/print_current_allowed_collision_matrix.py pr2_arm_navigation/pr2_arm_navigation_actions/scripts/print_current_allowed_collision_matrix.py
--- pr2_arm_navigation-0.4.8/pr2_arm_navigation_actions/scripts/print_current_allowed_collision_matrix.py	2012-01-06 20:23:20.000000000 +0100
+++ pr2_arm_navigation/pr2_arm_navigation_actions/scripts/print_current_allowed_collision_matrix.py	2012-05-08 14:35:27.370275290 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 import roslib; roslib.load_manifest('pr2_arm_navigation_actions')
 import rospy
diff -Naur pr2_arm_navigation-0.4.8/pr2_arm_navigation_actions/scripts/take_static_laser_collision_map.py pr2_arm_navigation/pr2_arm_navigation_actions/scripts/take_static_laser_collision_map.py
--- pr2_arm_navigation-0.4.8/pr2_arm_navigation_actions/scripts/take_static_laser_collision_map.py	2012-01-06 20:23:20.000000000 +0100
+++ pr2_arm_navigation/pr2_arm_navigation_actions/scripts/take_static_laser_collision_map.py	2012-05-08 14:35:27.366941957 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 import roslib; roslib.load_manifest('pr2_arm_navigation_actions')
 import rospy
