diff -Naur warehousewg-0.5.8/mongodb/src/wrapper.py warehousewg/mongodb/src/wrapper.py
--- warehousewg-0.5.8/mongodb/src/wrapper.py	2012-02-23 19:54:01.000000000 +0100
+++ warehousewg/mongodb/src/wrapper.py	2012-04-29 19:39:14.341792222 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 # This is a ros node wrapper for the mongod database server that
 # sets various things using ROS parameters:
diff -Naur warehousewg-0.5.8/mongo_ros/test/test_mongo_rospy.py warehousewg/mongo_ros/test/test_mongo_rospy.py
--- warehousewg-0.5.8/mongo_ros/test/test_mongo_rospy.py	2012-02-23 19:54:01.000000000 +0100
+++ warehousewg/mongo_ros/test/test_mongo_rospy.py	2012-04-29 19:39:14.345125555 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2009, Willow Garage, Inc.
