diff -Naur navigation-1.6.5/amcl/cfg/AMCL.cfg navigation/amcl/cfg/AMCL.cfg
--- navigation-1.6.5/amcl/cfg/AMCL.cfg	2011-10-27 00:03:03.000000000 +0200
+++ navigation/amcl/cfg/AMCL.cfg	2012-04-29 19:31:01.175116842 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PACKAGE = 'amcl'
 import roslib;roslib.load_manifest(PACKAGE)
diff -Naur navigation-1.6.5/amcl/test/basic_localization.py navigation/amcl/test/basic_localization.py
--- navigation-1.6.5/amcl/test/basic_localization.py	2011-10-27 00:03:03.000000000 +0200
+++ navigation/amcl/test/basic_localization.py	2012-04-29 19:31:01.171783509 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import roslib
 roslib.load_manifest('amcl')
diff -Naur navigation-1.6.5/amcl/test/set_pose.py navigation/amcl/test/set_pose.py
--- navigation-1.6.5/amcl/test/set_pose.py	2011-10-27 00:03:03.000000000 +0200
+++ navigation/amcl/test/set_pose.py	2012-04-29 19:31:01.168450176 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import roslib; roslib.load_manifest('amcl')
 import rospy
diff -Naur navigation-1.6.5/base_local_planner/cfg/BaseLocalPlanner.cfg navigation/base_local_planner/cfg/BaseLocalPlanner.cfg
--- navigation-1.6.5/base_local_planner/cfg/BaseLocalPlanner.cfg	2011-10-27 00:03:03.000000000 +0200
+++ navigation/base_local_planner/cfg/BaseLocalPlanner.cfg	2012-04-29 19:31:01.165116843 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PACKAGE='base_local_planner'
 import roslib; roslib.load_manifest(PACKAGE)
diff -Naur navigation-1.6.5/costmap_2d/cfg/Costmap2D.cfg navigation/costmap_2d/cfg/Costmap2D.cfg
--- navigation-1.6.5/costmap_2d/cfg/Costmap2D.cfg	2011-10-27 00:03:03.000000000 +0200
+++ navigation/costmap_2d/cfg/Costmap2D.cfg	2012-04-29 19:31:01.161783509 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PACKAGE='costmap_2d'
 import roslib; roslib.load_manifest(PACKAGE)
diff -Naur navigation-1.6.5/dwa_local_planner/cfg/DWAPlanner.cfg navigation/dwa_local_planner/cfg/DWAPlanner.cfg
--- navigation-1.6.5/dwa_local_planner/cfg/DWAPlanner.cfg	2011-10-27 00:03:03.000000000 +0200
+++ navigation/dwa_local_planner/cfg/DWAPlanner.cfg	2012-04-29 19:31:01.148450176 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # DWA Planner configuration
 
 PACKAGE='dwa_local_planner'
diff -Naur navigation-1.6.5/map_server/test/consumer.py navigation/map_server/test/consumer.py
--- navigation-1.6.5/map_server/test/consumer.py	2011-10-27 00:03:03.000000000 +0200
+++ navigation/map_server/test/consumer.py	2012-04-29 19:31:01.151783509 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur navigation-1.6.5/move_base/cfg/MoveBase.cfg navigation/move_base/cfg/MoveBase.cfg
--- navigation-1.6.5/move_base/cfg/MoveBase.cfg	2011-10-27 00:03:03.000000000 +0200
+++ navigation/move_base/cfg/MoveBase.cfg	2012-04-29 19:31:01.155116842 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PACKAGE = 'move_base'
 import roslib;roslib.load_manifest(PACKAGE)
diff -Naur navigation-1.6.5/robot_pose_ekf/scripts/wtf.py navigation/robot_pose_ekf/scripts/wtf.py
--- navigation-1.6.5/robot_pose_ekf/scripts/wtf.py	2011-10-27 00:03:03.000000000 +0200
+++ navigation/robot_pose_ekf/scripts/wtf.py	2012-04-29 19:31:01.158450175 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 import roslib; roslib.load_manifest('robot_pose_ekf')
 import sys
