diff -Naur arm_navigation_experimental-0.4.10/collider/scripts/construct_static_collision_map_test.py arm_navigation_experimental/collider/scripts/construct_static_collision_map_test.py
--- arm_navigation_experimental-0.4.10/collider/scripts/construct_static_collision_map_test.py	2011-11-22 01:41:53.000000000 +0100
+++ arm_navigation_experimental/collider/scripts/construct_static_collision_map_test.py	2012-04-29 20:36:12.181852605 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 import roslib; roslib.load_manifest('collider')
 import rospy
diff -Naur arm_navigation_experimental-0.4.10/collider/src/collider/construct_static_collision_map.py arm_navigation_experimental/collider/src/collider/construct_static_collision_map.py
--- arm_navigation_experimental-0.4.10/collider/src/collider/construct_static_collision_map.py	2011-11-22 01:41:53.000000000 +0100
+++ arm_navigation_experimental/collider/src/collider/construct_static_collision_map.py	2012-04-29 20:36:12.181852605 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 import roslib; roslib.load_manifest('collider')
 roslib.load_manifest('actionlib')
diff -Naur arm_navigation_experimental-0.4.10/distance_field/include/distance_field/voxel_grid.h arm_navigation_experimental/distance_field/include/distance_field/voxel_grid.h
--- arm_navigation_experimental-0.4.10/distance_field/include/distance_field/voxel_grid.h	2011-11-22 01:41:53.000000000 +0100
+++ arm_navigation_experimental/distance_field/include/distance_field/voxel_grid.h	2012-04-29 20:36:12.181852605 +0200
@@ -38,6 +38,7 @@
 #define DF_VOXEL_GRID_H_
 
 #include <algorithm>
+#include <math.h>
 
 namespace distance_field
 {
diff -Naur arm_navigation_experimental-0.4.10/move_arm_warehouse/CMakeLists.txt arm_navigation_experimental/move_arm_warehouse/CMakeLists.txt
--- arm_navigation_experimental-0.4.10/move_arm_warehouse/CMakeLists.txt	2011-11-22 01:41:53.000000000 +0100
+++ arm_navigation_experimental/move_arm_warehouse/CMakeLists.txt	2012-04-29 20:31:57.825181446 +0200
@@ -17,7 +17,7 @@
 
 find_package(Qt4 REQUIRED)
 include(${QT_USE_FILE})
-qt4_wrap_cpp(moc_sources ${QT_WIDGET_HEADERS})
+qt4_wrap_cpp(moc_sources ${QT_WIDGET_HEADERS} OPTIONS -DBOOST_TT_HAS_OPERATOR_HPP_INCLUDED)
 include_directories(${CMAKE_BINARY_DIR})
 include_directories(${QT_INCLUDE_DIR})
 
diff -Naur arm_navigation_experimental-0.4.10/move_arm_warehouse/include/move_arm_warehouse/planning_scene_warehouse_viewer.h arm_navigation_experimental/move_arm_warehouse/include/move_arm_warehouse/planning_scene_warehouse_viewer.h
--- arm_navigation_experimental-0.4.10/move_arm_warehouse/include/move_arm_warehouse/planning_scene_warehouse_viewer.h	2011-11-22 01:41:53.000000000 +0100
+++ arm_navigation_experimental/move_arm_warehouse/include/move_arm_warehouse/planning_scene_warehouse_viewer.h	2012-04-29 20:34:29.485184126 +0200
@@ -45,30 +45,30 @@
 
 #include <move_arm_warehouse/move_arm_utils.h>
 
-#include <qt4/QtGui/qwidget.h>
-#include <qt4/QtGui/qmenubar.h>
-#include <qt4/QtGui/qmenu.h>
-#include <qt4/QtGui/qprogressbar.h>
-#include <qt4/QtGui/qdialog.h>
-#include <qt4/QtGui/qtablewidget.h>
-#include <qt4/QtGui/qlayout.h>
-#include <qt4/QtGui/qpushbutton.h>
-#include <qt4/Qt/qthread.h>
-#include <qt4/QtGui/qslider.h>
-#include <qt4/QtGui/qspinbox.h>
-#include <qt4/QtGui/qlabel.h>
-#include <qt4/QtGui/qgroupbox.h>
-#include <qt4/QtGui/qcombobox.h>
-#include <qt4/QtGui/qcheckbox.h>
-#include <qt4/QtGui/qcolordialog.h>
-#include <qt4/QtGui/qtreewidget.h>
-#include <qt4/QtGui/qpalette.h>
-#include <qt4/QtGui/qformlayout.h>
-#include <qt4/QtGui/qlineedit.h>
-#include <qt4/QtGui/qfiledialog.h>
-#include <qt4/QtGui/qmessagebox.h>
-#include <qt4/QtGui/qmainwindow.h>
-#include <qt4/QtGui/qlistwidget.h>
+#include <QtGui/qwidget.h>
+#include <QtGui/qmenubar.h>
+#include <QtGui/qmenu.h>
+#include <QtGui/qprogressbar.h>
+#include <QtGui/qdialog.h>
+#include <QtGui/qtablewidget.h>
+#include <QtGui/qlayout.h>
+#include <QtGui/qpushbutton.h>
+#include <Qt/qthread.h>
+#include <QtGui/qslider.h>
+#include <QtGui/qspinbox.h>
+#include <QtGui/qlabel.h>
+#include <QtGui/qgroupbox.h>
+#include <QtGui/qcombobox.h>
+#include <QtGui/qcheckbox.h>
+#include <QtGui/qcolordialog.h>
+#include <QtGui/qtreewidget.h>
+#include <QtGui/qpalette.h>
+#include <QtGui/qformlayout.h>
+#include <QtGui/qlineedit.h>
+#include <QtGui/qfiledialog.h>
+#include <QtGui/qmessagebox.h>
+#include <QtGui/qmainwindow.h>
+#include <QtGui/qlistwidget.h>
 #include <QDateTime>
 
 static const std::string VIS_TOPIC_NAME = "planning_scene_visualizer_markers";
diff -Naur arm_navigation_experimental-0.4.10/move_arm_warehouse/scripts/create_launch_files.py arm_navigation_experimental/move_arm_warehouse/scripts/create_launch_files.py
--- arm_navigation_experimental-0.4.10/move_arm_warehouse/scripts/create_launch_files.py	2011-11-22 01:41:53.000000000 +0100
+++ arm_navigation_experimental/move_arm_warehouse/scripts/create_launch_files.py	2012-04-29 20:23:44.695172733 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 import sys
 import yaml
diff -Naur arm_navigation_experimental-0.4.10/move_arm_warehouse/src/planning_scene_warehouse_viewer.cpp arm_navigation_experimental/move_arm_warehouse/src/planning_scene_warehouse_viewer.cpp
--- arm_navigation_experimental-0.4.10/move_arm_warehouse/src/planning_scene_warehouse_viewer.cpp	2011-11-22 01:41:53.000000000 +0100
+++ arm_navigation_experimental/move_arm_warehouse/src/planning_scene_warehouse_viewer.cpp	2012-04-29 20:34:42.738517694 +0200
@@ -30,13 +30,13 @@
 // Author: Matthew Klingensmith, E. Gil Jones
 
 #include <move_arm_warehouse/planning_scene_warehouse_viewer.h>
-#include <qt4/QtGui/qapplication.h>
+#include <QtGui/qapplication.h>
 #include <assert.h>
-#include <qt4/QtGui/qsplashscreen.h>
-#include <qt4/QtGui/qdialogbuttonbox.h>
-#include <qt4/QtGui/qdialog.h>
-#include <qt4/QtGui/qheaderview.h>
-#include <qt4/QtGui/qimage.h>
+#include <QtGui/qsplashscreen.h>
+#include <QtGui/qdialogbuttonbox.h>
+#include <QtGui/qdialog.h>
+#include <QtGui/qheaderview.h>
+#include <QtGui/qimage.h>
 #include <ros/package.h>
 
 using namespace collision_space;
diff -Naur arm_navigation_experimental-0.4.10/move_arm_warehouse/test/testMotionExecutionBuffer.py arm_navigation_experimental/move_arm_warehouse/test/testMotionExecutionBuffer.py
--- arm_navigation_experimental-0.4.10/move_arm_warehouse/test/testMotionExecutionBuffer.py	2011-11-22 01:41:53.000000000 +0100
+++ arm_navigation_experimental/move_arm_warehouse/test/testMotionExecutionBuffer.py	2012-04-29 20:23:44.695172733 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'move_arm'
 
diff -Naur arm_navigation_experimental-0.4.10/move_arm_warehouse/test/testMovingOutOfAttachedContact.py arm_navigation_experimental/move_arm_warehouse/test/testMovingOutOfAttachedContact.py
--- arm_navigation_experimental-0.4.10/move_arm_warehouse/test/testMovingOutOfAttachedContact.py	2011-11-22 01:41:53.000000000 +0100
+++ arm_navigation_experimental/move_arm_warehouse/test/testMovingOutOfAttachedContact.py	2012-04-29 20:23:44.695172733 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'move_arm'
 
diff -Naur arm_navigation_experimental-0.4.10/move_arm_warehouse/test/testMovingOutOfTableContact.py arm_navigation_experimental/move_arm_warehouse/test/testMovingOutOfTableContact.py
--- arm_navigation_experimental-0.4.10/move_arm_warehouse/test/testMovingOutOfTableContact.py	2011-11-22 01:41:53.000000000 +0100
+++ arm_navigation_experimental/move_arm_warehouse/test/testMovingOutOfTableContact.py	2012-04-29 20:23:44.695172733 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'move_arm'
 
