diff -Naur arm_navigation-1.0.8/arm_navigation_msgs/src/arm_navigation_msgs/arm_navigation_msgs_utils.py arm_navigation/arm_navigation_msgs/src/arm_navigation_msgs/arm_navigation_msgs_utils.py
--- arm_navigation-1.0.8/arm_navigation_msgs/src/arm_navigation_msgs/arm_navigation_msgs_utils.py	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/arm_navigation_msgs/src/arm_navigation_msgs/arm_navigation_msgs_utils.py	2012-04-29 14:51:43.821487450 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'planning_environment'
 
diff -Naur arm_navigation-1.0.8/move_arm/test/testMotionExecutionBuffer.py arm_navigation/move_arm/test/testMotionExecutionBuffer.py
--- arm_navigation-1.0.8/move_arm/test/testMotionExecutionBuffer.py	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/move_arm/test/testMotionExecutionBuffer.py	2012-04-29 14:51:43.821487450 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'move_arm'
 
diff -Naur arm_navigation-1.0.8/move_arm/test/testMovingOutOfAttachedContact.py arm_navigation/move_arm/test/testMovingOutOfAttachedContact.py
--- arm_navigation-1.0.8/move_arm/test/testMovingOutOfAttachedContact.py	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/move_arm/test/testMovingOutOfAttachedContact.py	2012-04-29 14:51:43.821487450 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'move_arm'
 
diff -Naur arm_navigation-1.0.8/move_arm/test/testMovingOutOfTableContact.py arm_navigation/move_arm/test/testMovingOutOfTableContact.py
--- arm_navigation-1.0.8/move_arm/test/testMovingOutOfTableContact.py	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/move_arm/test/testMovingOutOfTableContact.py	2012-04-29 14:51:43.821487450 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'move_arm'
 
diff -Naur arm_navigation-1.0.8/ompl/Makefile arm_navigation/ompl/Makefile
--- arm_navigation-1.0.8/ompl/Makefile	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/ompl/Makefile	2012-04-29 14:52:13.654821309 +0200
@@ -2,15 +2,15 @@
 
 INSTALL_LOCATION=installation
 
-EXTRA_CMAKE_FLAGS=-DCMAKE_INSTALL_PREFIX=`pwd`/../$(INSTALL_LOCATION) -DOMPL_SOURCE:string="$(SOURCE_DIR)"
+EXTRA_CMAKE_FLAGS=-DCMAKE_INSTALL_PREFIX=`pwd`/../$(INSTALL_LOCATION) -DOMPL_SOURCE:string="$(SOURCE_DIR)" -DCMAKE_CXX_FLAGS=-D_POSIX_VERSION
 
-ompltoplevel:	all installed
+ompltoplevel: installed
 
 include $(shell rospack find mk)/cmake.mk
 
 clean: extra-clean
 
-installed:
+installed: all
 	cd build && make install
 	touch installed
 
diff -Naur arm_navigation-1.0.8/planning_environment/CMakeLists.txt arm_navigation/planning_environment/CMakeLists.txt
--- arm_navigation-1.0.8/planning_environment/CMakeLists.txt	2012-01-06 20:06:23.000000000 +0100
+++ arm_navigation/planning_environment/CMakeLists.txt	2012-04-29 14:51:43.821487450 +0200
@@ -69,7 +69,7 @@
 
 find_package(Qt4 REQUIRED)
 include(${QT_USE_FILE})
-qt4_wrap_cpp(moc_sources ${headers})
+qt4_wrap_cpp(moc_sources ${headers} OPTIONS -DBOOST_TT_HAS_OPERATOR_HPP_INCLUDED)
 include_directories(${CMAKE_BINARY_DIR})
 include_directories(${QT_INCLUDE_DIR})
 
diff -Naur arm_navigation-1.0.8/planning_environment/include/planning_environment/tools/planning_description_configuration_wizard.h arm_navigation/planning_environment/include/planning_environment/tools/planning_description_configuration_wizard.h
--- arm_navigation-1.0.8/planning_environment/include/planning_environment/tools/planning_description_configuration_wizard.h	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/planning_environment/include/planning_environment/tools/planning_description_configuration_wizard.h	2012-04-29 14:51:43.821487450 +0200
@@ -16,23 +16,23 @@
 #include <collision_space/environmentODE.h>
 #include <rosgraph_msgs/Clock.h>
 #include <planning_environment/util/collision_operations_generator.h>
-#include <qt4/QtGui/qwidget.h>
-#include <qt4/QtGui/qwizard.h>
-#include <qt4/QtGui/qlabel.h>
-#include <qt4/QtGui/qcombobox.h>
-#include <qt4/QtGui/qtreewidget.h>
-#include <qt4/QtGui/qtablewidget.h>
-#include <qt4/QtGui/qlineedit.h>
-#include <qt4/QtGui/qgridlayout.h>
-#include <qt4/QtGui/qgroupbox.h>
-#include <qt4/QtGui/qformlayout.h>
-#include <qt4/QtGui/qpushbutton.h>
-#include <qt4/QtGui/qcheckbox.h>
-#include <qt4/QtGui/qdialog.h>
-#include <qt4/QtGui/qfiledialog.h>
+#include <QtGui/qwidget.h>
+#include <QtGui/qwizard.h>
+#include <QtGui/qlabel.h>
+#include <QtGui/qcombobox.h>
+#include <QtGui/qtreewidget.h>
+#include <QtGui/qtablewidget.h>
+#include <QtGui/qlineedit.h>
+#include <QtGui/qgridlayout.h>
+#include <QtGui/qgroupbox.h>
+#include <QtGui/qformlayout.h>
+#include <QtGui/qpushbutton.h>
+#include <QtGui/qcheckbox.h>
+#include <QtGui/qdialog.h>
+#include <QtGui/qfiledialog.h>
 #include <tinyxml.h>
-#include <qt4/Qt/qthread.h>
-#include <qt4/QtGui/qprogressbar.h>
+#include <Qt/qthread.h>
+#include <QtGui/qprogressbar.h>
 
 static const std::string VIS_TOPIC_NAME = "planning_description_configuration_wizard";
 static const unsigned int CONTROL_SPEED = 10;
diff -Naur arm_navigation-1.0.8/planning_environment/scripts/add_attached_box.py arm_navigation/planning_environment/scripts/add_attached_box.py
--- arm_navigation-1.0.8/planning_environment/scripts/add_attached_box.py	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/planning_environment/scripts/add_attached_box.py	2012-04-29 14:51:43.821487450 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'planning_environment'
 
diff -Naur arm_navigation-1.0.8/planning_environment/src/tools/planning_description_configuration_wizard.cpp arm_navigation/planning_environment/src/tools/planning_description_configuration_wizard.cpp
--- arm_navigation-1.0.8/planning_environment/src/tools/planning_description_configuration_wizard.cpp	2012-01-06 20:06:23.000000000 +0100
+++ arm_navigation/planning_environment/src/tools/planning_description_configuration_wizard.cpp	2012-04-29 14:51:43.824820784 +0200
@@ -30,10 +30,10 @@
 // Author: E. Gil Jones
 
 #include <planning_environment/tools/planning_description_configuration_wizard.h>
-#include <qt4/QtGui/qapplication.h>
-#include <qt4/QtGui/qradiobutton.h>
-#include <qt4/QtGui/qdialogbuttonbox.h>
-#include <qt4/QtCore/qtextstream.h>
+#include <QtGui/qapplication.h>
+#include <QtGui/qradiobutton.h>
+#include <QtGui/qdialogbuttonbox.h>
+#include <QtCore/qtextstream.h>
 #include <QHeaderView>
 #include <boost/thread.hpp>
 
diff -Naur arm_navigation-1.0.8/planning_environment/test/add_pole.py arm_navigation/planning_environment/test/add_pole.py
--- arm_navigation-1.0.8/planning_environment/test/add_pole.py	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/planning_environment/test/add_pole.py	2012-04-29 14:51:43.824820784 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'planning_environment'
 
diff -Naur arm_navigation-1.0.8/planning_environment/test/fake_time.py arm_navigation/planning_environment/test/fake_time.py
--- arm_navigation-1.0.8/planning_environment/test/fake_time.py	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/planning_environment/test/fake_time.py	2012-04-29 14:51:43.824820784 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2010, Willow Garage, Inc.
diff -Naur arm_navigation-1.0.8/planning_environment/test/test_allowed_collision_operations.py arm_navigation/planning_environment/test/test_allowed_collision_operations.py
--- arm_navigation-1.0.8/planning_environment/test/test_allowed_collision_operations.py	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/planning_environment/test/test_allowed_collision_operations.py	2012-04-29 14:51:43.824820784 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 import roslib; roslib.load_manifest('planning_environment')
 import rospy
diff -Naur arm_navigation-1.0.8/planning_environment/test/test_alter_padding.py arm_navigation/planning_environment/test/test_alter_padding.py
--- arm_navigation-1.0.8/planning_environment/test/test_alter_padding.py	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/planning_environment/test/test_alter_padding.py	2012-04-29 14:51:43.824820784 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'planning_environment'
 
diff -Naur arm_navigation-1.0.8/planning_environment/test/test_attached_object_collisions.py arm_navigation/planning_environment/test/test_attached_object_collisions.py
--- arm_navigation-1.0.8/planning_environment/test/test_attached_object_collisions.py	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/planning_environment/test/test_attached_object_collisions.py	2012-04-29 14:51:43.824820784 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'planning_environment'
 
diff -Naur arm_navigation-1.0.8/planning_environment/test/test_collision_objects.py arm_navigation/planning_environment/test/test_collision_objects.py
--- arm_navigation-1.0.8/planning_environment/test/test_collision_objects.py	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/planning_environment/test/test_collision_objects.py	2012-04-29 14:51:43.824820784 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'planning_environment'
 
diff -Naur arm_navigation-1.0.8/planning_environment/test/test_get_base_state_validity.py arm_navigation/planning_environment/test/test_get_base_state_validity.py
--- arm_navigation-1.0.8/planning_environment/test/test_get_base_state_validity.py	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/planning_environment/test/test_get_base_state_validity.py	2012-04-29 14:51:43.824820784 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'planning_environment'
 
diff -Naur arm_navigation-1.0.8/planning_environment/test/test_get_current_state_validity.py arm_navigation/planning_environment/test/test_get_current_state_validity.py
--- arm_navigation-1.0.8/planning_environment/test/test_get_current_state_validity.py	2012-01-06 20:06:22.000000000 +0100
+++ arm_navigation/planning_environment/test/test_get_current_state_validity.py	2012-04-29 14:51:43.824820784 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 
 PKG = 'planning_environment'
 
