diff -Naur pr2_common-1.6.2/pr2_dashboard_aggregator/dashboard_aggregator.py pr2_common/pr2_dashboard_aggregator/dashboard_aggregator.py
--- pr2_common-1.6.2/pr2_dashboard_aggregator/dashboard_aggregator.py	2012-02-23 23:51:24.000000000 +0100
+++ pr2_common/pr2_dashboard_aggregator/dashboard_aggregator.py	2012-04-29 17:04:02.464961039 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 # Software License Agreement (BSD License)
 #
diff -Naur pr2_common-1.6.2/pr2_msgs/test/test_pr2_msgs_migration.py pr2_common/pr2_msgs/test/test_pr2_msgs_migration.py
--- pr2_common-1.6.2/pr2_msgs/test/test_pr2_msgs_migration.py	2012-02-23 23:51:24.000000000 +0100
+++ pr2_common/pr2_msgs/test/test_pr2_msgs_migration.py	2012-04-29 17:04:02.461627705 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
