diff -Naur xacro-1.6.0/src/xacro.py xacro/src/xacro.py
--- xacro-1.6.0/src/xacro.py	2011-06-30 00:53:28.000000000 +0200
+++ xacro/src/xacro.py	2012-04-17 20:05:14.535163298 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2008, Willow Garage, Inc.
 # All rights reserved.
 #
diff -Naur xacro-1.6.0/test/test_xacro.py xacro/test/test_xacro.py
--- xacro-1.6.0/test/test_xacro.py	2011-06-30 00:53:28.000000000 +0200
+++ xacro/test/test_xacro.py	2012-04-17 20:05:14.531829965 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 import roslib
 roslib.load_manifest('xacro')
 
diff -Naur xacro-1.6.0/xacro.py xacro/xacro.py
--- xacro-1.6.0/xacro.py	2011-06-30 00:53:28.000000000 +0200
+++ xacro/xacro.py	2012-04-17 20:05:14.528496632 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Copyright (c) 2008, Willow Garage, Inc.
 # All rights reserved.
 #
