diff --git a/src/bodies.cpp b/src/bodies.cpp
index 5e7facb..28d721f 100644
--- a/src/bodies.cpp
+++ b/src/bodies.cpp
@@ -41,14 +41,14 @@
 
 extern "C"
 {
-#include <qhull/qhull.h>
-#include <qhull/mem.h>
-#include <qhull/qset.h>
-#include <qhull/geom.h>
-#include <qhull/merge.h>
-#include <qhull/poly.h>
-#include <qhull/io.h>
-#include <qhull/stat.h>
+#include <libqhull/libqhull.h>
+#include <libqhull/mem.h>
+#include <libqhull/qset.h>
+#include <libqhull/geom.h>
+#include <libqhull/merge.h>
+#include <libqhull/poly.h>
+#include <libqhull/io.h>
+#include <libqhull/stat.h>
 }
 
 #include <boost/math/constants/constants.hpp>
