diff -Naur simulator_gazebo-1.4.13/gazebo/cfg/Physics.cfg simulator_gazebo/gazebo/cfg/Physics.cfg
--- simulator_gazebo-1.4.13/gazebo/cfg/Physics.cfg	2012-03-22 07:09:55.000000000 +0100
+++ simulator_gazebo/gazebo/cfg/Physics.cfg	2012-04-29 19:12:30.181763880 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 #*  All rights reserved.
 #*
 #*  Redistribution and use in source and binary forms, with or without
diff -Naur simulator_gazebo-1.4.13/gazebo/gazebo-arch.patch simulator_gazebo/gazebo/gazebo-arch.patch
--- simulator_gazebo-1.4.13/gazebo/gazebo-arch.patch	1970-01-01 01:00:00.000000000 +0100
+++ simulator_gazebo/gazebo/gazebo-arch.patch	2012-04-29 19:11:48.691763148 +0200
@@ -0,0 +1,55 @@
+diff -Naur gazebo-rbb429a3c12c1/server/gui/Gui.hh server/gui/Gui.hh
+--- gazebo-rbb429a3c12c1/server/gui/Gui.hh	2012-03-22 05:22:18.000000000 +0100
++++ server/gui/Gui.hh	2012-04-29 18:44:15.828400613 +0200
+@@ -30,6 +30,10 @@
+ #include <X11/Xlib.h>
+ #include <X11/Xutil.h>
+ 
++#ifndef Status
++#define Status int
++#endif
++
+ #include <FL/Fl.H>
+ #include <FL/Fl_Double_Window.H>
+ #include <FL/Fl_Slider.H>
+diff -Naur gazebo-rbb429a3c12c1/server/gui/ParamBrowser.hh server/gui/ParamBrowser.hh
+--- gazebo-rbb429a3c12c1/server/gui/ParamBrowser.hh	2012-03-22 05:22:18.000000000 +0100
++++ server/gui/ParamBrowser.hh	2012-04-29 18:45:55.755069045 +0200
+@@ -1,10 +1,10 @@
+ #ifndef PARAMBROWSER_HH
+ #define PARAMBROWSER_HH
+ 
+-#include <Fl/Fl_Scroll.H>
+-#include <Fl/Fl_Input.H>
+-#include <Fl/Fl_Output.H>
+-#include <Fl/fl_draw.H>
++#include <FL/Fl_Scroll.H>
++#include <FL/Fl_Input.H>
++#include <FL/Fl_Output.H>
++#include <FL/fl_draw.H>
+ #include <vector>
+ #include <string>
+ 
+diff -Naur gazebo-rbb429a3c12c1/server/rendering/CMakeLists.txt server/rendering/CMakeLists.txt
+--- gazebo-rbb429a3c12c1/server/rendering/CMakeLists.txt	2012-03-22 05:22:18.000000000 +0100
++++ server/rendering/CMakeLists.txt	2012-04-29 18:53:40.288410586 +0200
+@@ -60,6 +60,6 @@
+     ${CMAKE_LINK_FLAGS_${CMAKE_BUILD_TYPE}})
+ endif (CMAKE_LINK_FLAGS_${CMAKE_BUILD_TYPE})
+ 
+-target_link_libraries( gazebo_rendering ${ogre_libraries} xml2 X11)
++target_link_libraries( gazebo_rendering ${ogre_libraries} xml2 X11 GL)
+ install ( TARGETS gazebo_rendering DESTINATION ${CMAKE_INSTALL_PREFIX}/lib )
+ APPEND_TO_SERVER_HEADERS(${headers})
+diff -Naur gazebo-rbb429a3c12c1/server/rendering/OgreCreator.cc server/rendering/OgreCreator.cc
+--- gazebo-rbb429a3c12c1/server/rendering/OgreCreator.cc	2012-03-22 05:22:18.000000000 +0100
++++ server/rendering/OgreCreator.cc	2012-04-29 18:18:00.758372784 +0200
+@@ -23,6 +23,8 @@
+  * Date: 27 Dec 2007
+  */
+ 
++#define FL_LIBRARY
++
+ #include <Ogre.h>
+ 
+ #include <math.h>
diff -Naur simulator_gazebo-1.4.13/gazebo/Makefile.gazebo.tarball simulator_gazebo/gazebo/Makefile.gazebo.tarball
--- simulator_gazebo-1.4.13/gazebo/Makefile.gazebo.tarball	2012-03-22 07:09:55.000000000 +0100
+++ simulator_gazebo/gazebo/Makefile.gazebo.tarball	2012-04-29 19:11:58.051763313 +0200
@@ -5,7 +5,7 @@
 FILENAME = gazebo-rbb429a3c12c1.tgz
 TARBALL = build/$(FILENAME)
 TARBALL_URL = https://code.ros.org/svn/release/download/thirdparty/gazebo/$(FILENAME)
-TARBALL_PATCH = 
+TARBALL_PATCH = gazebo-arch.patch
 INITIAL_DIR = build/gazebo-rbb429a3c12c1
 SOURCE_DIR = build/gazebo
 UNPACK_CMD = tar xzf
diff -Naur simulator_gazebo-1.4.13/gazebo/scripts/spawn_model simulator_gazebo/gazebo/scripts/spawn_model
--- simulator_gazebo-1.4.13/gazebo/scripts/spawn_model	2012-03-22 07:09:55.000000000 +0100
+++ simulator_gazebo/gazebo/scripts/spawn_model	2012-04-29 19:12:30.181763880 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # 
 #  helper script for spawning models in gazebo
 # 
diff -Naur simulator_gazebo-1.4.13/gazebo/src/gazebo/gazebo_interface.py simulator_gazebo/gazebo/src/gazebo/gazebo_interface.py
--- simulator_gazebo-1.4.13/gazebo/src/gazebo/gazebo_interface.py	2012-03-22 07:09:55.000000000 +0100
+++ simulator_gazebo/gazebo/src/gazebo/gazebo_interface.py	2012-04-29 19:12:30.181763880 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Wrappers around the services provided by rosified gazebo
 
 import roslib; roslib.load_manifest('gazebo')
diff -Naur simulator_gazebo-1.4.13/gazebo_plugins/cfg/CameraSynchronizer.cfg simulator_gazebo/gazebo_plugins/cfg/CameraSynchronizer.cfg
--- simulator_gazebo-1.4.13/gazebo_plugins/cfg/CameraSynchronizer.cfg	2012-03-22 07:09:55.000000000 +0100
+++ simulator_gazebo/gazebo_plugins/cfg/CameraSynchronizer.cfg	2012-04-29 19:12:30.181763880 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 #*
 #*  Copyright (c) 2009, Willow Garage, Inc.
 #*  All rights reserved.
diff -Naur simulator_gazebo-1.4.13/gazebo_plugins/cfg/GazeboRosCamera.cfg simulator_gazebo/gazebo_plugins/cfg/GazeboRosCamera.cfg
--- simulator_gazebo-1.4.13/gazebo_plugins/cfg/GazeboRosCamera.cfg	2012-03-22 07:09:55.000000000 +0100
+++ simulator_gazebo/gazebo_plugins/cfg/GazeboRosCamera.cfg	2012-04-29 19:12:30.181763880 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PKG = 'gazebo_plugins'
 import roslib; roslib.load_manifest(PKG)
diff -Naur simulator_gazebo-1.4.13/gazebo_plugins/cfg/GazeboRosOpenniKinect.cfg simulator_gazebo/gazebo_plugins/cfg/GazeboRosOpenniKinect.cfg
--- simulator_gazebo-1.4.13/gazebo_plugins/cfg/GazeboRosOpenniKinect.cfg	2012-03-22 07:09:55.000000000 +0100
+++ simulator_gazebo/gazebo_plugins/cfg/GazeboRosOpenniKinect.cfg	2012-04-29 19:12:30.181763880 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 PKG = 'gazebo_plugins'
 import roslib; roslib.load_manifest(PKG)
diff -Naur simulator_gazebo-1.4.13/gazebo_plugins/cfg/Hokuyo.cfg simulator_gazebo/gazebo_plugins/cfg/Hokuyo.cfg
--- simulator_gazebo-1.4.13/gazebo_plugins/cfg/Hokuyo.cfg	2012-03-22 07:09:55.000000000 +0100
+++ simulator_gazebo/gazebo_plugins/cfg/Hokuyo.cfg	2012-04-29 19:12:30.181763880 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 #*  All rights reserved.
 #*
 #*  Redistribution and use in source and binary forms, with or without
diff -Naur simulator_gazebo-1.4.13/gazebo_plugins/scripts/gazebo_model simulator_gazebo/gazebo_plugins/scripts/gazebo_model
--- simulator_gazebo-1.4.13/gazebo_plugins/scripts/gazebo_model	2012-03-22 07:09:56.000000000 +0100
+++ simulator_gazebo/gazebo_plugins/scripts/gazebo_model	2012-04-29 19:12:30.181763880 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # 
 #  testing, do not use 
 # 
diff -Naur simulator_gazebo-1.4.13/gazebo_plugins/scripts/set_pose.py simulator_gazebo/gazebo_plugins/scripts/set_pose.py
--- simulator_gazebo-1.4.13/gazebo_plugins/scripts/set_pose.py	2012-03-22 07:09:56.000000000 +0100
+++ simulator_gazebo/gazebo_plugins/scripts/set_pose.py	2012-04-29 19:12:30.181763880 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur simulator_gazebo-1.4.13/gazebo_plugins/scripts/set_wrench.py simulator_gazebo/gazebo_plugins/scripts/set_wrench.py
--- simulator_gazebo-1.4.13/gazebo_plugins/scripts/set_wrench.py	2012-03-22 07:09:56.000000000 +0100
+++ simulator_gazebo/gazebo_plugins/scripts/set_wrench.py	2012-04-29 19:12:30.181763880 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Software License Agreement (BSD License)
 #
 # Copyright (c) 2008, Willow Garage, Inc.
diff -Naur simulator_gazebo-1.4.13/gazebo_plugins/src/gazebo_plugins/gazebo_plugins_interface.py simulator_gazebo/gazebo_plugins/src/gazebo_plugins/gazebo_plugins_interface.py
--- simulator_gazebo-1.4.13/gazebo_plugins/src/gazebo_plugins/gazebo_plugins_interface.py	2012-03-22 07:09:56.000000000 +0100
+++ simulator_gazebo/gazebo_plugins/src/gazebo_plugins/gazebo_plugins_interface.py	2012-04-29 19:12:30.181763880 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python2
 # Wrappers around the services provided by gazebo_ros_factory plugin
 
 import roslib; roslib.load_manifest('gazebo_plugins')
